{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import glob\n",
    "from bioutilities import Genome_2bit, Coordinate #https://github.com/lucapinello/bioutilities\n",
    "import os"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "#load genome sequence, here we use mm10\n",
    "genome=Genome_2bit('/homes10/lpinello/Haystack_dependencies/genomes/mm10.2bit') \n",
    "\n",
    "# you can download the mm10 genome in 2bit format from UCSC Genome Browser with\n",
    "# wget http://hgdownload.soe.ucsc.edu/goldenPath/mm10/bigZips/mm10.2bit"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#simple wrapper to perform a denovo motif analysis based on homer (http://homer.ucsd.edu/homer/motif/)\n",
    "def scan_denovo_homer(target_filename,bg_filename,window_size):\n",
    "    name='HOMER_DENOVO_%s_vs_%s' %(os.path.basename(target_filename),os.path.basename(bg_filename))\n",
    "    target_fasta=target_filename.replace('.bed','.%dbp.fa' %window_size)\n",
    "    bg_fasta=bg_filename.replace('.bed','.%dbp.fa' %window_size)\n",
    "    \n",
    "    target=Coordinate.bed_to_coordinates(target_filename)\n",
    "    background=Coordinate.bed_to_coordinates(bg_filename)\n",
    "    \n",
    "    target_window=Coordinate.coordinates_of_intervals_around_center(target,window_size)\n",
    "    background_window=Coordinate.coordinates_of_intervals_around_center(background,window_size)\n",
    "    \n",
    "    Coordinate.coordinates_to_fasta(target_window,target_fasta,genome)\n",
    "    Coordinate.coordinates_to_fasta(background_window,bg_fasta,genome)\n",
    "    !findMotifs.pl {target_fasta} fasta motifResults/{name} -fasta {bg_fasta} -noknown -cache 8000 \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "window_size=100 #100bp around the putative CRISPR SNPs\n",
    "bg_filename='GATK.dbSNP.bed' #as background we use common SNPs recovered in any mice with GATK"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "treatedFVB_normalF05.filt.bed\n",
      "\n",
      "Selected Options:\n",
      "\tInput file = treatedFVB_normalF05.filt.100bp.fa\n",
      "\tPromoter Set = fasta\n",
      "\tOutput Directory = motifResults/HOMER_DENOVO_treatedFVB_normalF05.filt.bed_vs_GATK.dbSNP.bed\n",
      "\tWill use FASTA files for motif finding\n",
      "\t\tTarget Sequences = treatedFVB_normalF05.filt.100bp.fa\n",
      "\t\tBackground Sequences = GATK.dbSNP.100bp.fa\n",
      "\tWill not search for known motifs\n",
      "\tUsing custom gene IDs for GO analysis\n",
      "\tParsing FASTA format files...\n",
      "\tFound 300 sequences\n",
      "\tFound 31078 sequences\n",
      "\n",
      "\tProgress: Step4 - removing redundant promoters\n",
      "\n",
      "\tProgress: Step5 - adjusting background sequences for GC/CpG content...\n",
      "\n",
      "\tSequences processed:\n",
      "\t\tAuto detected maximum sequence length of 101 bp\n",
      "\t\t31378 total\n",
      "\n",
      "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
      "\tFreq\tBin\tCount\n",
      "\t0.2\t0\t326\n",
      "\t0.25\t1\t788\n",
      "\t0.3\t2\t2374\n",
      "\t0.35\t3\t4459\n",
      "\t0.4\t4\t6204\n",
      "\t0.45\t5\t6033\n",
      "\t0.5\t6\t5636\n",
      "\t0.6\t7\t4906\n",
      "\t0.7\t8\t566\n",
      "\t0.8\t9\t82\n",
      "\t10\t10\t4\n",
      "\tBin\t# Targets\t# Background\tBackground Weight\n",
      "\t0\t4\t322\t1.287\n",
      "\t1\t9\t779\t1.197\n",
      "\t2\t22\t2352\t0.969\n",
      "\t3\t46\t4413\t1.080\n",
      "\t4\t44\t6160\t0.740\n",
      "\t5\t54\t5979\t0.936\n",
      "\t6\t66\t5570\t1.227\n",
      "\t7\t44\t4862\t0.937\n",
      "\t8\t8\t558\t1.485\n",
      "\t9\t2\t80\t2.590\n",
      "\t10\t1\t3\t34.531\n",
      "\n",
      "\tNormalizing lower order oligos using homer2\n",
      "\n",
      "\tReading input files...\n",
      "\t31378 total sequences read\n",
      "\tAutonormalization: 1-mers (4 total)\n",
      "\t\tA\t28.99%\t29.02%\t0.999\n",
      "\t\tC\t21.01%\t20.98%\t1.001\n",
      "\t\tG\t21.01%\t20.98%\t1.001\n",
      "\t\tT\t28.99%\t29.02%\t0.999\n",
      "\tAutonormalization: 2-mers (16 total)\n",
      "\t\tAA\t8.90%\t8.74%\t1.018\n",
      "\t\tCA\t7.20%\t7.42%\t0.971\n",
      "\t\tGA\t6.36%\t6.39%\t0.995\n",
      "\t\tTA\t6.52%\t6.46%\t1.008\n",
      "\t\tAC\t5.01%\t5.32%\t0.941\n",
      "\t\tCC\t5.43%\t5.23%\t1.038\n",
      "\t\tGC\t4.22%\t4.04%\t1.044\n",
      "\t\tTC\t6.36%\t6.39%\t0.995\n",
      "\t\tAG\t7.37%\t7.38%\t0.999\n",
      "\t\tCG\t1.02%\t0.95%\t1.069\n",
      "\t\tGG\t5.43%\t5.23%\t1.038\n",
      "\t\tTG\t7.20%\t7.42%\t0.971\n",
      "\t\tAT\t7.73%\t7.59%\t1.018\n",
      "\t\tCT\t7.37%\t7.38%\t0.999\n",
      "\t\tGT\t5.01%\t5.32%\t0.941\n",
      "\t\tTT\t8.90%\t8.74%\t1.018\n",
      "\tAutonormalization: 3-mers (64 total)\n",
      "\tNormalization weights can be found in file: motifResults/HOMER_DENOVO_treatedFVB_normalF05.filt.bed_vs_GATK.dbSNP.bed/seq.autonorm.tsv\n",
      "\tConverging on autonormalization solution:\n",
      "\t...............................................................................\n",
      "\tFinal normalization:\tAutonormalization: 1-mers (4 total)\n",
      "\t\tA\t28.99%\t28.99%\t1.000\n",
      "\t\tC\t21.01%\t21.01%\t1.000\n",
      "\t\tG\t21.01%\t21.01%\t1.000\n",
      "\t\tT\t28.99%\t28.99%\t1.000\n",
      "\tAutonormalization: 2-mers (16 total)\n",
      "\t\tAA\t8.90%\t8.89%\t1.001\n",
      "\t\tCA\t7.20%\t7.20%\t1.000\n",
      "\t\tGA\t6.36%\t6.37%\t0.999\n",
      "\t\tTA\t6.52%\t6.53%\t0.999\n",
      "\t\tAC\t5.01%\t5.02%\t0.999\n",
      "\t\tCC\t5.43%\t5.42%\t1.001\n",
      "\t\tGC\t4.22%\t4.21%\t1.001\n",
      "\t\tTC\t6.36%\t6.37%\t0.999\n",
      "\t\tAG\t7.37%\t7.37%\t1.000\n",
      "\t\tCG\t1.02%\t1.01%\t1.007\n",
      "\t\tGG\t5.43%\t5.42%\t1.001\n",
      "\t\tTG\t7.20%\t7.20%\t1.000\n",
      "\t\tAT\t7.73%\t7.72%\t1.001\n",
      "\t\tCT\t7.37%\t7.37%\t1.000\n",
      "\t\tGT\t5.01%\t5.02%\t0.999\n",
      "\t\tTT\t8.90%\t8.89%\t1.001\n",
      "\tAutonormalization: 3-mers (64 total)\n",
      "\n",
      "\tProgress: Step6 - Gene Ontology Enrichment Analysis\n",
      "\tSkipping...\n",
      "\n",
      "\tProgress: Step7 - Known motif enrichment\n",
      "\tSkipping...\n",
      "\n",
      "\tProgress: Step8 - De novo motif finding (HOMER)\n",
      "\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 32427\n",
      "\tAutoadjustment for sequence coverage in background: 1.00x\n",
      "\n",
      "\tOligos: 32427 of 34497 max\n",
      "\tTree  : 66248 of 172485 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 1 mismatches...\n",
      "\n",
      "\tScreening oligos 32427 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t54.43% skipped, 45.57% checked (14778 of 32427), of those checked:\n",
      "\t\t54.43% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 32427 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t54.43% skipped, 45.57% checked (14778 of 32427), of those checked:\n",
      "\t\t0.00% not in target, 22.88% increased p-value, 30.55% high p-value\n",
      "\tReading input files...\n",
      "\t31378 total sequences read\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: GATGGATG... (-20.530)\n",
      "\t\tRound 1: -20.81 GATGGATG T:34.0(10.73%),B:929.7(2.97%),P:1e-9\n",
      "\t\tRound 2: -22.12 GATGGATG T:77.0(22.67%),B:3294.8(10.12%),P:1e-9\n",
      "\t\tRound 3: -24.43 GAASKAWG T:94.0(26.94%),B:4080.6(12.38%),P:1e-10\n",
      "\t\tRound 4: -24.43 GAASKAWG T:94.0(26.94%),B:4080.6(12.38%),P:1e-10\n",
      "\t\t=Final=: -4.29 GAASKAWG T:43.0(14.33%),B:3129.8(10.14%),P:1e-1\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: TCTATTCT... (-17.591)\n",
      "\t\tRound 1: -23.50 TCTATTCT T:38.0(11.92%),B:986.6(3.14%),P:1e-10\n",
      "\t\tRound 2: -23.50 TCTATTCT T:38.0(11.92%),B:986.6(3.14%),P:1e-10\n",
      "\t\t=Final=: -6.74 TCTATTCT T:18.0(6.00%),B:808.7(2.62%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: AGAGGCTA... (-9.236)\n",
      "\t\tRound 1: -12.61 AGAGGCTA T:11.0(3.61%),B:149.6(0.48%),P:1e-5\n",
      "\t\tRound 2: -12.61 AGAGGCTA T:11.0(3.61%),B:149.6(0.48%),P:1e-5\n",
      "\t\t=Final=: -14.81 AGAGGCTA T:11.0(3.67%),B:145.7(0.47%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: GTTGTGCC... (-8.536)\n",
      "\t\tRound 1: -10.17 GTTGTGCC T:25.0(8.01%),B:984.6(3.14%),P:1e-4\n",
      "\t\tRound 2: -10.17 GTTGTGCC T:25.0(8.01%),B:984.6(3.14%),P:1e-4\n",
      "\t\t=Final=: -10.20 GTTGTGCC T:24.0(8.00%),B:966.8(3.13%),P:1e-4\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: TATATAGA... (-7.992)\n",
      "\t\tRound 1: -13.14 TATATAGA T:173.0(43.88%),B:11353.1(30.78%),P:1e-5\n",
      "\t\tRound 2: -13.33 TATAKAGA T:185.0(46.08%),B:12283.6(32.83%),P:1e-5\n",
      "\t\tRound 3: -13.33 TATAKAGA T:185.0(46.08%),B:12283.6(32.83%),P:1e-5\n",
      "\t\t=Final=: -5.19 TATAKAGA T:6.0(2.00%),B:160.9(0.52%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\tRemaining seeds don't look promising (After initial 5 motifs, logp -7.559 > -8.477)\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t31378 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing hypergeometric scoring\n",
      "\tChecking enrichment of 5 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: motifResults/HOMER_DENOVO_treatedFVB_normalF05.filt.bed_vs_GATK.dbSNP.bed/homerMotifs.motifs8\n",
      "\n",
      "\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 355030\n",
      "\tAutoadjustment for sequence coverage in background: 1.00x\n",
      "\n",
      "\tOligos: 355030 of 551887 max\n",
      "\tTree  : 766280 of 2759435 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 1 mismatches...\n",
      "\n",
      "\tScreening oligos 355030 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t93.24% skipped, 6.76% checked (24007 of 355030), of those checked:\n",
      "\t\t93.24% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 355030 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t93.24% skipped, 6.76% checked (24007 of 355030), of those checked:\n",
      "\t\t0.00% not in target, 6.17% increased p-value, 0.68% high p-value\n",
      "\tReading input files...\n",
      "\t31378 total sequences read\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: GAATAGAATA... (-27.565)\n",
      "\t\tRound 1: -29.82 GAATAGAATA T:14.0(4.57%),B:57.5(0.18%),P:1e-12\n",
      "\t\tRound 2: -32.28 TAATAGAMTA T:16.0(5.20%),B:76.1(0.25%),P:1e-14\n",
      "\t\tRound 3: -32.28 TAATAGAMTA T:16.0(5.20%),B:76.1(0.25%),P:1e-14\n",
      "\t\t=Final=: -9.20 TAATAGAMTA T:3.0(1.00%),B:7.9(0.03%),P:1e-3\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: AATAGAATAG... (-26.342)\n",
      "\t\tRound 1: -33.11 AATAGAATAG T:15.0(4.89%),B:56.8(0.18%),P:1e-14\n",
      "\t\tRound 2: -33.11 AATAGAATAG T:15.0(4.89%),B:56.8(0.18%),P:1e-14\n",
      "\t\t=Final=: -6.13 AATAGAATAG T:3.0(1.00%),B:24.2(0.08%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: GATGGATGGA... (-26.058)\n",
      "\t\tRound 1: -30.81 GATGGATGGA T:66.0(19.78%),B:2077.9(6.51%),P:1e-13\n",
      "\t\tRound 2: -33.38 GATGGATGGA T:78.0(22.93%),B:2535.0(7.88%),P:1e-14\n",
      "\t\tRound 3: -35.32 GAAGRATGGA T:87.0(25.21%),B:2907.9(8.99%),P:1e-15\n",
      "\t\tRound 4: -37.11 GAAGAATGGA T:91.0(26.20%),B:3015.6(9.31%),P:1e-16\n",
      "\t\tRound 5: -37.11 GAAGAATGGA T:91.0(26.20%),B:3015.6(9.31%),P:1e-16\n",
      "\t\t=Final=: -13.18 GAAGAATGGA T:30.0(10.00%),B:1164.3(3.77%),P:1e-5\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: TATATAGATA... (-12.037)\n",
      "\t\tRound 1: -17.62 TATATAGATA T:152.0(39.80%),B:8902.4(25.06%),P:1e-7\n",
      "\t\tRound 2: -17.67 TATATAGATA T:150.0(39.40%),B:8774.4(24.74%),P:1e-7\n",
      "\t\tRound 3: -17.81 TATATAGAKA T:158.0(40.99%),B:9230.4(25.85%),P:1e-7\n",
      "\t\tRound 4: -17.81 TATATAGAKA T:158.0(40.99%),B:9230.4(25.85%),P:1e-7\n",
      "\t\t=Final=: -8.28 TATATAGAKA T:6.0(2.00%),B:85.6(0.28%),P:1e-3\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: CTCTTCTCTT... (-9.976)\n",
      "\t\tRound 1: -17.15 CTCTTCTCTT T:14.0(4.57%),B:174.5(0.56%),P:1e-7\n",
      "\t\tRound 2: -19.39 CYCTYCTCTG T:18.0(5.83%),B:274.8(0.88%),P:1e-8\n",
      "\t\tRound 3: -20.52 CYCTHCTCTA T:20.0(6.46%),B:326.7(1.05%),P:1e-8\n",
      "\t\tRound 4: -20.52 CYCTHCTCTA T:20.0(6.46%),B:326.7(1.05%),P:1e-8\n",
      "\t\t=Final=: -15.55 CYCTHCTCTA T:14.0(4.67%),B:237.5(0.77%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: GCTGCCCCTC... (-9.849)\n",
      "\t\tRound 1: -18.66 GCTGCCCCTC T:25.0(8.01%),B:594.1(1.91%),P:1e-8\n",
      "\t\tRound 2: -18.66 GCTGCCCCTC T:25.0(8.01%),B:594.1(1.91%),P:1e-8\n",
      "\t\t=Final=: -19.54 GCTGCCCCTC T:26.0(8.67%),B:656.8(2.13%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: AGGAACCTGC... (-9.927)\n",
      "\t\tRound 1: -25.59 AGGAACCTGC T:24.0(7.70%),B:375.9(1.21%),P:1e-11\n",
      "\t\tRound 2: -25.59 AGGAACCTGC T:24.0(7.70%),B:375.9(1.21%),P:1e-11\n",
      "\t\t=Final=: -23.92 AGGAACCTGC T:22.0(7.33%),B:368.4(1.19%),P:1e-10\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t8 of 25 Initial Sequence: ATCAGAATGG... (-9.552)\n",
      "\t\tRound 1: -11.46 ATCAGAATGG T:5.0(1.66%),B:12.5(0.04%),P:1e-4\n",
      "\t\tRound 2: -13.47 AGCAGAATGG T:23.0(7.39%),B:685.8(2.19%),P:1e-5\n",
      "\t\tRound 3: -13.47 AGCAGAATGG T:23.0(7.39%),B:685.8(2.19%),P:1e-5\n",
      "\t\t=Final=: -14.82 AGCAGAATGG T:23.0(7.67%),B:675.2(2.19%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t9 of 25 Initial Sequence: ATGATGGCTT... (-9.391)\n",
      "\t\tRound 1: -16.36 ATGATGGCTT T:12.0(3.93%),B:124.6(0.40%),P:1e-7\n",
      "\t\tRound 2: -16.36 ATGATGGCTT T:12.0(3.93%),B:124.6(0.40%),P:1e-7\n",
      "\t\t=Final=: -18.54 ATGATGGCTT T:12.0(4.00%),B:124.6(0.40%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t10 of 25 Initial Sequence: CACCATCAGT... (-9.290)\n",
      "\t\tRound 1: -9.55 CACCATCAGT T:4.0(1.33%),B:7.9(0.02%),P:1e-4\n",
      "\t\tRound 2: -9.55 CACCATCAGT T:4.0(1.33%),B:7.9(0.02%),P:1e-4\n",
      "\t\t=Final=: -13.94 CACCATCAGT T:3.0(1.00%),B:0.9(0.00%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t11 of 25 Initial Sequence: AGAGAGAGAC... (-9.122)\n",
      "\t\tRound 1: -12.66 AGAGAGAGAC T:11.0(3.61%),B:148.2(0.48%),P:1e-5\n",
      "\t\tRound 2: -12.81 AGAGKKACAC T:24.0(7.70%),B:773.4(2.47%),P:1e-5\n",
      "\t\tRound 3: -14.14 WGAGTGACMC T:16.0(5.20%),B:312.2(1.01%),P:1e-6\n",
      "\t\tRound 4: -14.14 WGAGTGACMC T:16.0(5.20%),B:312.2(1.01%),P:1e-6\n",
      "\t\t=Final=: -10.84 WGAGTGACMC T:12.0(4.00%),B:268.9(0.87%),P:1e-4\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t12 of 25 Initial Sequence: TAGATAGATA... (-8.903)\n",
      "\t\tRound 1: -10.19 TABATAGATA T:131.0(35.43%),B:8825.8(24.87%),P:1e-4\n",
      "\t\tRound 2: -10.19 TABATAGATA T:131.0(35.43%),B:8825.8(24.87%),P:1e-4\n",
      "\t\t=Final=: -3.70 TABATAGATA T:17.0(5.67%),B:1024.3(3.32%),P:1e-1\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t13 of 25 Initial Sequence: TGATGATGAT... (-8.849)\n",
      "\t\tRound 1: -13.20 TGATGATGAT T:9.0(2.96%),B:79.0(0.26%),P:1e-5\n",
      "\t\tRound 2: -13.32 WGATGATSAT T:11.0(3.61%),B:137.7(0.44%),P:1e-5\n",
      "\t\tRound 3: -13.32 WGATGATSAT T:11.0(3.61%),B:137.7(0.44%),P:1e-5\n",
      "\t\t=Final=: -3.20 WGATGATSAT T:3.0(1.00%),B:77.0(0.25%),P:1e-1\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\tRemaining seeds don't look promising (After initial 5 motifs, logp -8.217 > -8.477)\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t31378 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing hypergeometric scoring\n",
      "\tChecking enrichment of 13 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: motifResults/HOMER_DENOVO_treatedFVB_normalF05.filt.bed_vs_GATK.dbSNP.bed/homerMotifs.motifs10\n",
      "\n",
      "\n",
      "\t-blen automatically set to 2\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 1214517\n",
      "\tAutoadjustment for sequence coverage in background: 1.00x\n",
      "\n",
      "\tOligos: 1214517 of 2824020 max\n",
      "\tTree  : 3135080 of 14120100 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 1 mismatches...\n",
      "\n",
      "\tScreening oligos 1214517 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t97.95% skipped, 2.05% checked (24905 of 1214517), of those checked:\n",
      "\t\t97.95% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 1214517 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t97.95% skipped, 2.05% checked (24905 of 1214517), of those checked:\n",
      "\t\t0.00% not in target, 1.97% increased p-value, 0.08% high p-value\n",
      "\tReading input files...\n",
      "\t31378 total sequences read\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: TCGAATAGAATA... (-29.041)\n",
      "\t\tRound 1: -32.98 TAGAATAAAATA T:14.0(4.57%),B:43.1(0.14%),P:1e-14\n",
      "\t\tRound 2: -32.98 TAGAATAAAATA T:14.0(4.57%),B:43.1(0.14%),P:1e-14\n",
      "\t\t=Final=: -4.72 TAGAATAAAATA T:2.0(0.67%),B:13.1(0.04%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: ATAGATGGATGG... (-28.502)\n",
      "\t\tRound 1: -34.80 AAAGCTGGATGG T:32.0(10.13%),B:458.0(1.47%),P:1e-15\n",
      "\t\tRound 2: -34.91 AWKGMTGKATRG T:33.0(10.43%),B:491.6(1.58%),P:1e-15\n",
      "\t\tRound 3: -34.97 AWAGMTGGATGG T:32.0(10.13%),B:454.3(1.46%),P:1e-15\n",
      "\t\tRound 4: -36.21 ATGGCTGTATGG T:34.0(10.73%),B:504.2(1.62%),P:1e-15\n",
      "\t\tRound 5: -38.91 AAGCCTGTATGG T:38.0(11.92%),B:564.4(1.81%),P:1e-16\n",
      "\t\tRound 6: -38.91 AAGCCTGTATGG T:38.0(11.92%),B:564.4(1.81%),P:1e-16\n",
      "\t\t=Final=: -9.58 AAGCCTGTATGG T:6.0(2.00%),B:66.0(0.21%),P:1e-4\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: ATGGATGGATGG... (-27.989)\n",
      "\t\tRound 1: -32.24 ATGGATGGATGG T:28.0(8.93%),B:362.1(1.17%),P:1e-13\n",
      "\t\tRound 2: -32.24 ATGGATGGATGG T:28.0(8.93%),B:362.1(1.17%),P:1e-13\n",
      "\t\t=Final=: -3.54 ATGGATGGATGG T:3.0(1.00%),B:66.2(0.21%),P:1e-1\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: CTTTTCTCTTCT... (-19.781)\n",
      "\t\tRound 1: -22.91 CTYTTCTCTTCT T:23.0(7.39%),B:392.7(1.26%),P:1e-9\n",
      "\t\tRound 2: -24.67 CTCTTCTCTTCT T:30.0(9.53%),B:602.1(1.93%),P:1e-10\n",
      "\t\tRound 3: -30.49 CTYTTCTCTTCT T:35.0(11.03%),B:670.5(2.15%),P:1e-13\n",
      "\t\tRound 4: -31.52 CTCTTCTTTTST T:38.0(11.92%),B:731.2(2.34%),P:1e-13\n",
      "\t\tRound 5: -31.52 CTCTTCTTTTST T:38.0(11.92%),B:731.2(2.34%),P:1e-13\n",
      "\t\t=Final=: -6.67 CTCTTCTTTTST T:10.0(3.33%),B:314.8(1.02%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: TTCCTTCCTCCC... (-17.854)\n",
      "\t\tRound 1: -22.48 TTCCTYCCTCCC T:116.0(32.11%),B:5728.6(16.94%),P:1e-9\n",
      "\t\tRound 2: -30.01 TYCCTCCYTCCC T:93.0(26.69%),B:3627.8(11.09%),P:1e-13\n",
      "\t\tRound 3: -30.01 TYCCTCCYTCCC T:93.0(26.69%),B:3627.8(11.09%),P:1e-13\n",
      "\t\t=Final=: -2.96 TYCCTCCYTCCC T:112.0(37.33%),B:10097.1(32.71%),P:1e-1\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: CTTCCTTCCTTC... (-63.989)\n",
      "\t\tRound 1: -4.64 CTTCCTTSCTTC T:1.0(0.33%),B:0.0(0.00%),P:1e-2\n",
      "\t\tRound 2: -8.94 TTTVCTTBCTTC T:6.0(1.98%),B:47.5(0.15%),P:1e-3\n",
      "\t\tRound 3: -11.94 TTTACTAKGTTM T:6.0(1.98%),B:24.2(0.08%),P:1e-5\n",
      "\t\tRound 4: -17.22 YTTVCTTTCTTA T:14.0(4.57%),B:173.3(0.56%),P:1e-7\n",
      "\t\tRound 5: -17.22 YTTVCTTTCTTA T:14.0(4.57%),B:173.3(0.56%),P:1e-7\n",
      "\t\t=Final=: -19.39 YTTVCTTTCTTA T:14.0(4.67%),B:171.3(0.55%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: TCTTTCTTTCTT... (-52.847)\n",
      "\t\tRound 1: -6.06 TCTTTCTTTCTT T:65.0(19.51%),B:4412.5(13.32%),P:1e-2\n",
      "\t\tRound 2: -6.06 TCTTTCTTTCTT T:65.0(19.51%),B:4412.5(13.32%),P:1e-2\n",
      "\t\t=Final=: -1.30 TCTTTCTTTCTT T:52.0(17.33%),B:4912.3(15.91%),P:1e0\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t8 of 25 Initial Sequence: GATGATGATTAT... (-16.673)\n",
      "\t\tRound 1: -22.31 GATGATGACTAT T:11.0(3.61%),B:49.7(0.16%),P:1e-9\n",
      "\t\tRound 2: -23.86 GATGAGTACTAT T:12.0(3.93%),B:57.8(0.18%),P:1e-10\n",
      "\t\tRound 3: -25.00 GATGATTAYWAT T:11.0(3.61%),B:36.9(0.12%),P:1e-10\n",
      "\t\tRound 4: -25.00 GATGATTAYWAT T:11.0(3.61%),B:36.9(0.12%),P:1e-10\n",
      "\t\t=Final=: -13.80 GATGATTAYWAT T:4.0(1.33%),B:5.1(0.02%),P:1e-5\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t9 of 25 Initial Sequence: GATCCTCCAATA... (-12.968)\n",
      "\t\tRound 1: -17.78 GATCCTCCAAYA T:11.0(3.61%),B:82.6(0.27%),P:1e-7\n",
      "\t\tRound 2: -22.74 GAWCCTCCAACA T:10.0(3.28%),B:32.1(0.10%),P:1e-9\n",
      "\t\tRound 3: -22.74 GAWCCTCCAACA T:10.0(3.28%),B:32.1(0.10%),P:1e-9\n",
      "\t\t=Final=: -20.65 GAWCCTCCAACA T:7.0(2.33%),B:15.4(0.05%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t10 of 25 Initial Sequence: CAATCAGAATGG... (-12.561)\n",
      "\t\tRound 1: -12.66 CAATCAGAATGG T:11.0(3.61%),B:148.7(0.48%),P:1e-5\n",
      "\t\tRound 2: -29.97 CAWTCAGRTTGG T:10.0(3.28%),B:12.8(0.04%),P:1e-13\n",
      "\t\tRound 3: -29.97 CAWTCAGRTTGG T:10.0(3.28%),B:12.8(0.04%),P:1e-13\n",
      "\t\t=Final=: -22.78 CAWTCAGRTTGG T:7.0(2.33%),B:11.0(0.04%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t11 of 25 Initial Sequence: AAATTAATTTTT... (-11.212)\n",
      "\t\tRound 1: -22.30 AAATTAATTTTT T:8.0(2.64%),B:12.6(0.04%),P:1e-9\n",
      "\t\tRound 2: -22.30 AAATTAATTTTT T:8.0(2.64%),B:12.6(0.04%),P:1e-9\n",
      "\t\t=Final=: -21.84 AAATTAATTTTT T:7.0(2.33%),B:12.6(0.04%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t12 of 25 Initial Sequence: TATATATAGCTA... (-9.888)\n",
      "\t\tRound 1: -14.95 TATATATAGCTA T:6.0(1.98%),B:12.3(0.04%),P:1e-6\n",
      "\t\tRound 2: -14.95 TATATATAGCTA T:6.0(1.98%),B:12.3(0.04%),P:1e-6\n",
      "\t\t=Final=: -7.51 TATATATAGCTA T:2.0(0.67%),B:2.5(0.01%),P:1e-3\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t13 of 25 Initial Sequence: ACACCGGTTCCG... (-9.290)\n",
      "\t\tRound 1: -16.99 ACSCYGGTTCCK T:5.0(1.66%),B:2.8(0.01%),P:1e-7\n",
      "\t\tRound 2: -18.59 ACVCCGGTTCCG T:5.0(1.66%),B:0.6(0.00%),P:1e-8\n",
      "\t\tRound 3: -18.59 ACVCCGGTTCCG T:5.0(1.66%),B:0.6(0.00%),P:1e-8\n",
      "\t\t=Final=: -4.64 ACVCCGGTTCCG T:1.0(0.33%),B:0.0(0.00%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t14 of 25 Initial Sequence: GTGTGGGGGGGG... (-8.943)\n",
      "\t\tRound 1: -18.08 GTGTGSGGGGGG T:15.0(4.89%),B:192.3(0.62%),P:1e-7\n",
      "\t\tRound 2: -22.75 GTGBGBGGGGGG T:22.0(7.08%),B:357.4(1.15%),P:1e-9\n",
      "\t\tRound 3: -22.94 GTGKGBGGGGGG T:20.0(6.46%),B:280.8(0.90%),P:1e-9\n",
      "\t\tRound 4: -22.94 GTGKGBGGGGGG T:20.0(6.46%),B:280.8(0.90%),P:1e-9\n",
      "\t\t=Final=: -5.72 GTGKGBGGGGGG T:25.0(8.33%),B:1410.1(4.57%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t15 of 25 Initial Sequence: AGAACTCATTGT... (-8.891)\n",
      "\t\tRound 1: -12.59 AGAACTCATTGT T:8.0(2.64%),B:60.3(0.19%),P:1e-5\n",
      "\t\tRound 2: -12.59 AGAACTCATTGT T:8.0(2.64%),B:60.3(0.19%),P:1e-5\n",
      "\t\t=Final=: -15.02 AGAACTCATTGT T:8.0(2.67%),B:60.3(0.20%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t16 of 25 Initial Sequence: GGTCAAATGCTT... (-8.848)\n",
      "\t\tRound 1: -13.97 GGTCAAAKGCTT T:8.0(2.64%),B:48.2(0.16%),P:1e-6\n",
      "\t\tRound 2: -15.58 GGTCAAAGGCTT T:8.0(2.64%),B:38.0(0.12%),P:1e-6\n",
      "\t\tRound 3: -15.58 GGTCAAAGGCTT T:8.0(2.64%),B:38.0(0.12%),P:1e-6\n",
      "\t\t=Final=: -18.36 GGTCAAAGGCTT T:8.0(2.67%),B:38.0(0.12%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\tRemaining seeds don't look promising (After initial 5 motifs, logp -8.198 > -8.477)\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t31378 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing hypergeometric scoring\n",
      "\tChecking enrichment of 16 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: motifResults/HOMER_DENOVO_treatedFVB_normalF05.filt.bed_vs_GATK.dbSNP.bed/homerMotifs.motifs12\n",
      "\n",
      "\t(Motifs in homer2 format)\n",
      "\tDetermining similar motifs... 34 reduced to 18 motifs\n",
      "\tOutputing HTML and sequence logos for motif comparison...\n",
      "\tChecking de novo motifs against known motifs...\n",
      "\tFormatting HTML page...\n",
      "\t\t1 of 18 (1e-10) similar to TEIL(AP2/EREBP)/Nicotiana tabacum/AthaMap(0.695)\n",
      "\t\t2 of 18 (1e-9) similar to YML081W(MacIsaac)/Yeast(0.673)\n",
      "\t\t3 of 18 (1e-9) similar to MZF1(var.2)/MA0057.1/Jaspar(0.720)\n",
      "\t\t4 of 18 (1e-9) similar to Lhx3/MA0135.1/Jaspar(0.806)\n",
      "\t\t5 of 18 (1e-8) similar to HOXA2(Homeobox)/mES-Hoxa2-ChIP-Seq(Donaldson_et_al.)/Homer(0.697)\n",
      "\t\t6 of 18 (1e-7) similar to Nr2e3/MA0164.1/Jaspar(0.662)\n",
      "\t\t7 of 18 (1e-6) similar to RNP4F(RRM)/Drosophila_melanogaster-RNCMPT00060-PBM/HughesRNA(0.775)\n",
      "\t\t8 of 18 (1e-6) similar to CRZ1(MacIsaac)/Yeast(0.683)\n",
      "\t\t9 of 18 (1e-6) similar to PGR(NR)/EndoStromal-PGR-ChIP-Seq(GSE69539)/Homer(0.738)\n",
      "\t\t10 of 18 (1e-5) similar to RBM47(RRM)/Gallus_gallus-RNCMPT00279-PBM/HughesRNA(0.706)\n",
      "\t\t11 of 18 (1e-5) similar to TEC1/TEC1_YPD/[](Harbison)/Yeast(0.756)\n",
      "\t\t12 of 18 (1e-4) similar to z/dmmpmm(Down)/fly(0.769)\n",
      "\t\t13 of 18 (1e-4) similar to PHD1(MacIsaac)/Yeast(0.801)\n",
      "\t\t14 of 18 (1e-4) similar to RAP1/MA0359.1/Jaspar(0.609)\n",
      "\t\t15 of 18 (1e-3) similar to prd-HD/dmmpmm(Bergman)/fly(0.720)\n",
      "\t\t16 of 18 (1e-3) similar to SeqBias: TA-repeat(0.842)\n",
      "\t\t17 of 18 (1e-2) similar to Zfp281(Zf)/ES-Zfp281-ChIP-Seq(GSE81042)/Homer(0.813)\n",
      "\t\t18 of 18 (1e-1) similar to GAGA-repeat/SacCer-Promoters/Homer(0.712)\n",
      "\tJob finished\n",
      "\n",
      "treatedF03_normalFVB.filt.bed\n",
      "\n",
      "Selected Options:\n",
      "\tInput file = treatedF03_normalFVB.filt.100bp.fa\n",
      "\tPromoter Set = fasta\n",
      "\tOutput Directory = motifResults/HOMER_DENOVO_treatedF03_normalFVB.filt.bed_vs_GATK.dbSNP.bed\n",
      "\tWill use FASTA files for motif finding\n",
      "\t\tTarget Sequences = treatedF03_normalFVB.filt.100bp.fa\n",
      "\t\tBackground Sequences = GATK.dbSNP.100bp.fa\n",
      "\tWill not search for known motifs\n",
      "\tUsing custom gene IDs for GO analysis\n",
      "\tParsing FASTA format files...\n",
      "\tFound 1861 sequences\n",
      "\tFound 31078 sequences\n",
      "\n",
      "\tProgress: Step4 - removing redundant promoters\n",
      "\n",
      "\tProgress: Step5 - adjusting background sequences for GC/CpG content...\n",
      "\n",
      "\tSequences processed:\n",
      "\t\tAuto detected maximum sequence length of 101 bp\n",
      "\t\t32939 total\n",
      "\n",
      "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
      "\tFreq\tBin\tCount\n",
      "\t0.2\t0\t339\n",
      "\t0.25\t1\t843\n",
      "\t0.3\t2\t2481\n",
      "\t0.35\t3\t4645\n",
      "\t0.4\t4\t6475\n",
      "\t0.45\t5\t6352\n",
      "\t0.5\t6\t5925\n",
      "\t0.6\t7\t5189\n",
      "\t0.7\t8\t593\n",
      "\t0.8\t9\t92\n",
      "\t10\t10\t5\n",
      "\tBin\t# Targets\t# Background\tBackground Weight\n",
      "\t0\t17\t322\t0.882\n",
      "\t1\t64\t779\t1.372\n",
      "\t2\t129\t2352\t0.916\n",
      "\t3\t232\t4413\t0.878\n",
      "\t4\t315\t6160\t0.854\n",
      "\t5\t373\t5979\t1.042\n",
      "\t6\t355\t5570\t1.064\n",
      "\t7\t327\t4862\t1.123\n",
      "\t8\t35\t558\t1.047\n",
      "\t9\t12\t80\t2.505\n",
      "\t10\t2\t3\t11.133\n",
      "\n",
      "\tNormalizing lower order oligos using homer2\n",
      "\n",
      "\tReading input files...\n",
      "\t32939 total sequences read\n",
      "\tAutonormalization: 1-mers (4 total)\n",
      "\t\tA\t28.93%\t28.94%\t1.000\n",
      "\t\tC\t21.07%\t21.06%\t1.000\n",
      "\t\tG\t21.07%\t21.06%\t1.000\n",
      "\t\tT\t28.93%\t28.94%\t1.000\n",
      "\tAutonormalization: 2-mers (16 total)\n",
      "\t\tAA\t9.00%\t8.68%\t1.037\n",
      "\t\tCA\t7.21%\t7.46%\t0.966\n",
      "\t\tGA\t6.31%\t6.42%\t0.983\n",
      "\t\tTA\t6.40%\t6.36%\t1.006\n",
      "\t\tAC\t5.15%\t5.35%\t0.963\n",
      "\t\tCC\t5.35%\t5.26%\t1.019\n",
      "\t\tGC\t4.25%\t4.04%\t1.050\n",
      "\t\tTC\t6.31%\t6.42%\t0.983\n",
      "\t\tAG\t7.39%\t7.43%\t0.995\n",
      "\t\tCG\t1.12%\t0.91%\t1.229\n",
      "\t\tGG\t5.35%\t5.26%\t1.019\n",
      "\t\tTG\t7.21%\t7.46%\t0.966\n",
      "\t\tAT\t7.38%\t7.49%\t0.985\n",
      "\t\tCT\t7.39%\t7.43%\t0.995\n",
      "\t\tGT\t5.15%\t5.35%\t0.963\n",
      "\t\tTT\t9.00%\t8.68%\t1.037\n",
      "\tAutonormalization: 3-mers (64 total)\n",
      "\tNormalization weights can be found in file: motifResults/HOMER_DENOVO_treatedF03_normalFVB.filt.bed_vs_GATK.dbSNP.bed/seq.autonorm.tsv\n",
      "\tConverging on autonormalization solution:\n",
      "\t...............................................................................\n",
      "\tFinal normalization:\tAutonormalization: 1-mers (4 total)\n",
      "\t\tA\t28.93%\t28.93%\t1.000\n",
      "\t\tC\t21.07%\t21.07%\t1.000\n",
      "\t\tG\t21.07%\t21.07%\t1.000\n",
      "\t\tT\t28.93%\t28.93%\t1.000\n",
      "\tAutonormalization: 2-mers (16 total)\n",
      "\t\tAA\t9.00%\t9.00%\t1.000\n",
      "\t\tCA\t7.21%\t7.21%\t1.001\n",
      "\t\tGA\t6.31%\t6.32%\t0.999\n",
      "\t\tTA\t6.40%\t6.40%\t1.001\n",
      "\t\tAC\t5.15%\t5.16%\t0.999\n",
      "\t\tCC\t5.35%\t5.36%\t0.999\n",
      "\t\tGC\t4.25%\t4.24%\t1.001\n",
      "\t\tTC\t6.31%\t6.32%\t0.999\n",
      "\t\tAG\t7.39%\t7.39%\t1.001\n",
      "\t\tCG\t1.12%\t1.11%\t1.004\n",
      "\t\tGG\t5.35%\t5.36%\t0.999\n",
      "\t\tTG\t7.21%\t7.21%\t1.001\n",
      "\t\tAT\t7.38%\t7.39%\t0.998\n",
      "\t\tCT\t7.39%\t7.39%\t1.001\n",
      "\t\tGT\t5.15%\t5.16%\t0.999\n",
      "\t\tTT\t9.00%\t9.00%\t1.000\n",
      "\tAutonormalization: 3-mers (64 total)\n",
      "\n",
      "\tProgress: Step6 - Gene Ontology Enrichment Analysis\n",
      "\tSkipping...\n",
      "\n",
      "\tProgress: Step7 - Known motif enrichment\n",
      "\tSkipping...\n",
      "\n",
      "\tProgress: Step8 - De novo motif finding (HOMER)\n",
      "\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 32509\n",
      "\tAutoadjustment for sequence coverage in background: 1.00x\n",
      "\n",
      "\tOligos: 32509 of 34497 max\n",
      "\tTree  : 66732 of 172485 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 1 mismatches...\n",
      "\n",
      "\tScreening oligos 32509 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t16.11% skipped, 83.89% checked (27273 of 32509), of those checked:\n",
      "\t\t16.11% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 32509 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t16.11% skipped, 83.89% checked (27273 of 32509), of those checked:\n",
      "\t\t0.00% not in target, 33.68% increased p-value, 53.49% high p-value\n",
      "\tReading input files...\n",
      "\t32939 total sequences read\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: GGGGGGGG... (-21.623)\n",
      "\t\tRound 1: -34.57 GGGGGGGG T:322.0(15.89%),B:3137.2(9.73%),P:1e-15\n",
      "\t\tRound 2: -34.92 GGGGGGGG T:292.0(14.53%),B:2767.1(8.63%),P:1e-15\n",
      "\t\tRound 3: -34.92 GGGGGGGG T:292.0(14.53%),B:2767.1(8.63%),P:1e-15\n",
      "\t\t=Final=: -12.38 GGGGGGGG T:110.0(5.91%),B:1135.9(3.71%),P:1e-5\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: TTCCTTCC... (-14.054)\n",
      "\t\tRound 1: -18.85 TTCCTTCC T:241.0(12.15%),B:2605.2(8.15%),P:1e-8\n",
      "\t\tRound 2: -19.29 TGMCTWCV T:294.0(14.62%),B:3291.9(10.18%),P:1e-8\n",
      "\t\tRound 3: -21.13 TTCCTTCC T:301.0(14.94%),B:3298.2(10.20%),P:1e-9\n",
      "\t\tRound 4: -21.13 TTCCTTCC T:301.0(14.94%),B:3298.2(10.20%),P:1e-9\n",
      "\t\t=Final=: -10.25 TTCCTTCC T:179.0(9.62%),B:2157.9(7.04%),P:1e-4\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: GCGAGTCT... (-13.606)\n",
      "\t\tRound 1: -18.30 GCGAGTCT T:68.0(3.59%),B:488.0(1.58%),P:1e-7\n",
      "\t\tRound 2: -18.30 GCGAGTCT T:68.0(3.59%),B:488.0(1.58%),P:1e-7\n",
      "\t\t=Final=: -18.38 GCGAGTCT T:66.0(3.55%),B:482.7(1.58%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: AATCAATA... (-10.345)\n",
      "\t\tRound 1: -10.35 AATCAATA T:24.0(1.28%),B:135.2(0.44%),P:1e-4\n",
      "\t\tRound 2: -10.35 AATCAATA T:24.0(1.28%),B:135.2(0.44%),P:1e-4\n",
      "\t\t=Final=: -11.68 AATCAATA T:24.0(1.29%),B:131.5(0.43%),P:1e-5\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: CTTTTCAG... (-9.977)\n",
      "\t\tRound 1: -23.07 CTTTTCAG T:187.0(9.56%),B:1779.8(5.64%),P:1e-10\n",
      "\t\tRound 2: -26.52 CTTTTCAG T:210.0(10.67%),B:1978.2(6.25%),P:1e-11\n",
      "\t\tRound 3: -26.52 CTTTTCAG T:210.0(10.67%),B:1978.2(6.25%),P:1e-11\n",
      "\t\t=Final=: -30.57 CTTTTCAG T:205.0(11.02%),B:1913.6(6.24%),P:1e-13\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: GAAAGGTC... (-10.368)\n",
      "\t\tRound 1: -10.92 GAAAGGTC T:25.0(1.33%),B:140.0(0.45%),P:1e-4\n",
      "\t\tRound 2: -10.92 GAAAGGTC T:25.0(1.33%),B:140.0(0.45%),P:1e-4\n",
      "\t\t=Final=: -11.93 GAAAGGTC T:25.0(1.34%),B:138.4(0.45%),P:1e-5\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: TAGAATAG... (-9.746)\n",
      "\t\tRound 1: -15.91 TAGAATAG T:56.0(2.97%),B:400.4(1.30%),P:1e-6\n",
      "\t\tRound 2: -17.45 TAGAAWAC T:121.0(6.30%),B:1120.5(3.59%),P:1e-7\n",
      "\t\tRound 3: -17.45 TAGAAWAC T:121.0(6.30%),B:1120.5(3.59%),P:1e-7\n",
      "\t\t=Final=: -11.46 TAGAAWAC T:103.0(5.53%),B:1069.1(3.49%),P:1e-4\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\tRemaining seeds don't look promising (After initial 5 motifs, logp -9.266 > -9.270)\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t32939 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing hypergeometric scoring\n",
      "\tChecking enrichment of 7 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: motifResults/HOMER_DENOVO_treatedF03_normalFVB.filt.bed_vs_GATK.dbSNP.bed/homerMotifs.motifs8\n",
      "\n",
      "\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 362816\n",
      "\tAutoadjustment for sequence coverage in background: 1.00x\n",
      "\n",
      "\tOligos: 362816 of 551887 max\n",
      "\tTree  : 781464 of 2759435 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 1 mismatches...\n",
      "\n",
      "\tScreening oligos 362816 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t68.13% skipped, 31.87% checked (115620 of 362816), of those checked:\n",
      "\t\t68.13% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 362816 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t68.13% skipped, 31.87% checked (115620 of 362816), of those checked:\n",
      "\t\t0.00% not in target, 22.17% increased p-value, 15.10% high p-value\n",
      "\tReading input files...\n",
      "\t32939 total sequences read\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: CCATCCTTCC... (-21.375)\n",
      "\t\tRound 1: -28.45 CCWTCCTTCC T:149.0(7.70%),B:1223.9(3.91%),P:1e-12\n",
      "\t\tRound 2: -31.13 CCTTBCWTCC T:180.0(9.22%),B:1521.6(4.84%),P:1e-13\n",
      "\t\tRound 3: -32.59 CCWYNCWTSC T:181.0(9.27%),B:1505.9(4.79%),P:1e-14\n",
      "\t\tRound 4: -32.59 CCWYNCWTSC T:181.0(9.27%),B:1505.9(4.79%),P:1e-14\n",
      "\t\t=Final=: -10.70 CCWYNCWTSC T:57.0(3.06%),B:505.3(1.65%),P:1e-4\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: TACATATACA... (-18.466)\n",
      "\t\tRound 1: -20.88 TACATATACA T:38.0(2.02%),B:173.7(0.56%),P:1e-9\n",
      "\t\tRound 2: -23.39 GACATATASA T:50.0(2.65%),B:259.9(0.84%),P:1e-10\n",
      "\t\tRound 3: -27.12 TACATATAGA T:43.0(2.28%),B:172.7(0.56%),P:1e-11\n",
      "\t\tRound 4: -28.02 TACATATAGA T:42.0(2.23%),B:159.8(0.52%),P:1e-12\n",
      "\t\tRound 5: -28.02 TACATATAGA T:42.0(2.23%),B:159.8(0.52%),P:1e-12\n",
      "\t\t=Final=: -8.67 TACATATAGA T:11.0(0.59%),B:42.5(0.14%),P:1e-3\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: TCTCTAAAAC... (-18.004)\n",
      "\t\tRound 1: -19.94 TCTCTAAAAC T:12.0(0.64%),B:10.1(0.03%),P:1e-8\n",
      "\t\tRound 2: -19.94 TCTCTAAAAC T:12.0(0.64%),B:10.1(0.03%),P:1e-8\n",
      "\t\t=Final=: -21.52 TCTCTAAAAC T:12.0(0.64%),B:10.1(0.03%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: ATACATATAC... (-17.718)\n",
      "\t\tRound 1: -21.10 ATACATATAC T:33.0(1.76%),B:132.0(0.43%),P:1e-9\n",
      "\t\tRound 2: -21.10 ATACATATAC T:33.0(1.76%),B:132.0(0.43%),P:1e-9\n",
      "\t\t=Final=: -5.08 ATACATATAC T:12.0(0.64%),B:80.8(0.26%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: ATCTATTCTA... (-16.197)\n",
      "\t\tRound 1: -25.53 ATCTATTCTA T:73.0(3.85%),B:450.6(1.46%),P:1e-11\n",
      "\t\tRound 2: -28.14 ATCTATTCTA T:37.0(1.97%),B:122.9(0.40%),P:1e-12\n",
      "\t\tRound 3: -28.14 ATCTATTCTA T:37.0(1.97%),B:122.9(0.40%),P:1e-12\n",
      "\t\t=Final=: -18.32 ATCTATTCTA T:25.0(1.34%),B:93.5(0.31%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: GAAATCAATA... (-15.365)\n",
      "\t\tRound 1: -21.78 GAAATCAATA T:23.0(1.23%),B:58.7(0.19%),P:1e-9\n",
      "\t\tRound 2: -21.78 GAAATCAATA T:23.0(1.23%),B:58.7(0.19%),P:1e-9\n",
      "\t\t=Final=: -23.42 GAAATCAATA T:23.0(1.24%),B:57.8(0.19%),P:1e-10\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: TATAGGTTTG... (-15.273)\n",
      "\t\tRound 1: -20.83 TATAGGTTTG T:13.0(0.70%),B:12.9(0.04%),P:1e-9\n",
      "\t\tRound 2: -20.83 TATAGGTTTG T:13.0(0.70%),B:12.9(0.04%),P:1e-9\n",
      "\t\t=Final=: -23.51 TATAGGTTTG T:9.0(0.48%),B:1.3(0.00%),P:1e-10\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t8 of 25 Initial Sequence: AATGCCCAAG... (-15.273)\n",
      "\t\tRound 1: -15.40 AATGCCCAAG T:8.0(0.43%),B:4.4(0.01%),P:1e-6\n",
      "\t\tRound 2: -15.40 AATGCCCAAG T:8.0(0.43%),B:4.4(0.01%),P:1e-6\n",
      "\t\t=Final=: -16.90 AATGCCCAAG T:8.0(0.43%),B:4.4(0.01%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t9 of 25 Initial Sequence: TGGGGGGGGG... (-14.060)\n",
      "\t\tRound 1: -28.32 NGGGGGGGGN T:314.0(15.53%),B:3233.8(10.01%),P:1e-12\n",
      "\t\tRound 2: -31.11 DGGGGGGGGN T:254.0(12.76%),B:2400.4(7.53%),P:1e-13\n",
      "\t\tRound 3: -31.11 DGGGGGGGGN T:254.0(12.76%),B:2400.4(7.53%),P:1e-13\n",
      "\t\t=Final=: -12.87 DGGGGGGGGN T:103.0(5.53%),B:1031.6(3.37%),P:1e-5\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t10 of 25 Initial Sequence: CGCCGCAGCC... (-13.389)\n",
      "\t\tRound 1: -24.49 CGCCGCAGCC T:12.0(0.64%),B:5.4(0.02%),P:1e-10\n",
      "\t\tRound 2: -24.49 CGCCGCAGCC T:12.0(0.64%),B:5.4(0.02%),P:1e-10\n",
      "\t\t=Final=: -12.89 CGCCGCAGCC T:7.0(0.38%),B:6.5(0.02%),P:1e-5\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t11 of 25 Initial Sequence: AAATATATAT... (-13.314)\n",
      "\t\tRound 1: -15.14 AAATATATAT T:441.0(21.10%),B:5504.1(16.44%),P:1e-6\n",
      "\t\tRound 2: -15.14 AAATATATAT T:441.0(21.10%),B:5504.1(16.44%),P:1e-6\n",
      "\t\t=Final=: -5.80 AAATATATAT T:42.0(2.26%),B:428.9(1.40%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t12 of 25 Initial Sequence: TGTGTGTGTG... (-12.935)\n",
      "\t\tRound 1: -15.42 TGTGTGTGTG T:287.0(14.30%),B:3360.6(10.39%),P:1e-6\n",
      "\t\tRound 2: -16.93 AGTGWGTGAG T:314.0(15.53%),B:3661.9(11.26%),P:1e-7\n",
      "\t\tRound 3: -20.16 AGCGTGTGWG T:331.0(16.30%),B:3745.2(11.50%),P:1e-8\n",
      "\t\tRound 4: -20.16 AGCGTGTGWG T:331.0(16.30%),B:3745.2(11.50%),P:1e-8\n",
      "\t\t=Final=: -4.87 AGCGTGTGWG T:75.0(4.03%),B:911.2(2.97%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t13 of 25 Initial Sequence: TTTACCCTGC... (-12.563)\n",
      "\t\tRound 1: -13.61 TTTACCCTGC T:8.0(0.43%),B:6.4(0.02%),P:1e-5\n",
      "\t\tRound 2: -13.61 TTTACCCTGC T:8.0(0.43%),B:6.4(0.02%),P:1e-5\n",
      "\t\t=Final=: -15.20 TTTACCCTGC T:8.0(0.43%),B:6.4(0.02%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t14 of 25 Initial Sequence: GGGCTTTTCT... (-12.507)\n",
      "\t\tRound 1: -23.04 GGGCTTTTCT T:27.0(1.44%),B:78.4(0.25%),P:1e-10\n",
      "\t\tRound 2: -24.51 GGGCTYTTCT T:43.0(2.28%),B:189.4(0.61%),P:1e-10\n",
      "\t\tRound 3: -25.28 GGGSTYTTCT T:37.0(1.97%),B:137.1(0.45%),P:1e-10\n",
      "\t\tRound 4: -25.28 GGGSTYTTCT T:37.0(1.97%),B:137.1(0.45%),P:1e-10\n",
      "\t\t=Final=: -25.29 GGGSTYTTCT T:36.0(1.93%),B:136.2(0.44%),P:1e-10\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t15 of 25 Initial Sequence: GAAGATGATG... (-12.029)\n",
      "\t\tRound 1: -22.10 GAAGATGATG T:40.0(2.13%),B:181.3(0.59%),P:1e-9\n",
      "\t\tRound 2: -22.11 GAAGATGATG T:45.0(2.39%),B:224.3(0.73%),P:1e-9\n",
      "\t\tRound 3: -22.57 GWAGATGATG T:43.0(2.28%),B:203.3(0.66%),P:1e-9\n",
      "\t\tRound 4: -22.57 GWAGATGATG T:43.0(2.28%),B:203.3(0.66%),P:1e-9\n",
      "\t\t=Final=: -17.01 GWAGATGATG T:34.0(1.83%),B:176.7(0.58%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t16 of 25 Initial Sequence: AAACAGTATT... (-11.853)\n",
      "\t\tRound 1: -26.02 AAACAGTATT T:85.0(4.47%),B:569.7(1.84%),P:1e-11\n",
      "\t\tRound 2: -29.37 AAACAGTATT T:98.0(5.13%),B:654.0(2.11%),P:1e-12\n",
      "\t\tRound 3: -31.62 AAACAGTATT T:85.0(4.47%),B:505.6(1.63%),P:1e-13\n",
      "\t\tRound 4: -33.85 AAACAGTATT T:71.0(3.74%),B:352.3(1.14%),P:1e-14\n",
      "\t\tRound 5: -33.85 AAACAGTATT T:71.0(3.74%),B:352.3(1.14%),P:1e-14\n",
      "\t\t=Final=: -33.50 AAACAGTATT T:68.0(3.65%),B:344.0(1.12%),P:1e-14\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t17 of 25 Initial Sequence: CCCCCACCCC... (-11.671)\n",
      "\t\tRound 1: -3.15 NVCCCRCCNS T:6.0(0.32%),B:30.3(0.10%),P:1e-1\n",
      "\t\tRound 2: -6.57 WWCMSRTCKR T:117.0(6.09%),B:1409.6(4.49%),P:1e-2\n",
      "\t\tRound 3: -7.53 TTMASATCTG T:286.0(14.25%),B:3790.0(11.63%),P:1e-3\n",
      "\t\tRound 4: -9.54 WTMASATCTG T:73.0(3.85%),B:714.2(2.30%),P:1e-4\n",
      "\t\tRound 5: -11.50 WDMASATCTG T:49.0(2.60%),B:388.6(1.26%),P:1e-4\n",
      "\t\tRound 6: -12.78 TWMASATCTG T:111.0(5.79%),B:1105.8(3.54%),P:1e-5\n",
      "\t\tRound 7: -12.78 TWMASATCTG T:111.0(5.79%),B:1105.8(3.54%),P:1e-5\n",
      "\t\t=Final=: -14.53 TWMASATCTG T:104.0(5.59%),B:1003.6(3.27%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t18 of 25 Initial Sequence: GCCCTTTCAG... (-11.444)\n",
      "\t\tRound 1: -13.94 GCCCTTTCAG T:7.0(0.38%),B:3.9(0.01%),P:1e-6\n",
      "\t\tRound 2: -19.40 GCCGTTTCAG T:10.0(0.54%),B:5.2(0.02%),P:1e-8\n",
      "\t\tRound 3: -19.40 GCCGTTTCAG T:10.0(0.54%),B:5.2(0.02%),P:1e-8\n",
      "\t\t=Final=: -20.88 GCCGTTTCAG T:10.0(0.54%),B:5.2(0.02%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t19 of 25 Initial Sequence: TCTGCTGAAC... (-11.444)\n",
      "\t\tRound 1: -23.34 TCTGCTGAAC T:15.0(0.80%),B:15.9(0.05%),P:1e-10\n",
      "\t\tRound 2: -26.82 TCTGCTGAAT T:23.0(1.23%),B:42.4(0.14%),P:1e-11\n",
      "\t\tRound 3: -26.82 TCTGCTGAAT T:23.0(1.23%),B:42.4(0.14%),P:1e-11\n",
      "\t\t=Final=: -28.67 TCTGCTGAAT T:23.0(1.24%),B:41.8(0.14%),P:1e-12\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t20 of 25 Initial Sequence: TGATGTAGTA... (-11.444)\n",
      "\t\tRound 1: -23.13 TGATGTAGTA T:11.0(0.59%),B:4.4(0.01%),P:1e-10\n",
      "\t\tRound 2: -23.13 TGATGTAGTA T:11.0(0.59%),B:4.4(0.01%),P:1e-10\n",
      "\t\t=Final=: -24.59 TGATGTAGTA T:13.0(0.70%),B:9.3(0.03%),P:1e-10\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t21 of 25 Initial Sequence: TGGTGGCAAA... (-11.359)\n",
      "\t\tRound 1: -19.12 TGGTGGCAAA T:11.0(0.59%),B:8.7(0.03%),P:1e-8\n",
      "\t\tRound 2: -20.19 TGGTGGCWAA T:13.0(0.70%),B:13.2(0.04%),P:1e-8\n",
      "\t\tRound 3: -20.19 TGGTGGCWAA T:13.0(0.70%),B:13.2(0.04%),P:1e-8\n",
      "\t\t=Final=: -21.77 TGGTGGCWAA T:13.0(0.70%),B:13.2(0.04%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t22 of 25 Initial Sequence: CTGTCCAGAA... (-10.873)\n",
      "\t\tRound 1: -18.40 CTGTCCAGAA T:27.0(1.44%),B:101.7(0.33%),P:1e-7\n",
      "\t\tRound 2: -19.94 CTGTCCAGAA T:38.0(2.02%),B:180.6(0.59%),P:1e-8\n",
      "\t\tRound 3: -20.47 HTGTCCAGAR T:38.0(2.02%),B:176.3(0.57%),P:1e-8\n",
      "\t\tRound 4: -20.47 HTGTCCAGAR T:38.0(2.02%),B:176.3(0.57%),P:1e-8\n",
      "\t\t=Final=: -21.64 HTGTCCAGAR T:38.0(2.04%),B:175.7(0.57%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t23 of 25 Initial Sequence: TAATCTCAAT... (-10.689)\n",
      "\t\tRound 1: -18.69 TAATCTCAAT T:48.0(2.55%),B:284.9(0.92%),P:1e-8\n",
      "\t\tRound 2: -18.69 TAATCTCAAT T:48.0(2.55%),B:284.9(0.92%),P:1e-8\n",
      "\t\t=Final=: -19.20 TAATCTCAAT T:43.0(2.31%),B:239.5(0.78%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t24 of 25 Initial Sequence: TGAGGGCATA... (-10.427)\n",
      "\t\tRound 1: -14.31 TGAGGGCATA T:10.0(0.54%),B:12.4(0.04%),P:1e-6\n",
      "\t\tRound 2: -18.49 TGAGRKCATA T:13.0(0.70%),B:16.3(0.05%),P:1e-8\n",
      "\t\tRound 3: -18.49 TGAGRKCATA T:13.0(0.70%),B:16.3(0.05%),P:1e-8\n",
      "\t\t=Final=: -20.06 TGAGRKCATA T:13.0(0.70%),B:16.3(0.05%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t25 of 25 Initial Sequence: AGCTTCCCCT... (-10.427)\n",
      "\t\tRound 1: -19.67 AGCTTCCCCT T:15.0(0.80%),B:23.0(0.07%),P:1e-8\n",
      "\t\tRound 2: -19.67 AGCTTCCCCT T:15.0(0.80%),B:23.0(0.07%),P:1e-8\n",
      "\t\t=Final=: -21.20 AGCTTCCCCT T:15.0(0.81%),B:23.0(0.07%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t32939 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing hypergeometric scoring\n",
      "\tChecking enrichment of 25 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: motifResults/HOMER_DENOVO_treatedF03_normalFVB.filt.bed_vs_GATK.dbSNP.bed/homerMotifs.motifs10\n",
      "\n",
      "\n",
      "\t-blen automatically set to 2\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 1286755\n",
      "\tAutoadjustment for sequence coverage in background: 1.00x\n",
      "\n",
      "\tOligos: 1286755 of 2964510 max\n",
      "\tTree  : 3303880 of 14822550 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 1 mismatches...\n",
      "\n",
      "\tScreening oligos 1286755 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t88.64% skipped, 11.36% checked (146221 of 1286755), of those checked:\n",
      "\t\t88.64% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 1286755 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t88.64% skipped, 11.36% checked (146221 of 1286755), of those checked:\n",
      "\t\t0.00% not in target, 10.23% increased p-value, 0.90% high p-value\n",
      "\tReading input files...\n",
      "\t32939 total sequences read\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: AAGAAATCAATA... (-24.621)\n",
      "\t\tRound 1: -34.81 AAGAAATCAATA T:19.0(1.02%),B:12.2(0.04%),P:1e-15\n",
      "\t\tRound 2: -34.81 AAGAAATCAATA T:19.0(1.02%),B:12.2(0.04%),P:1e-15\n",
      "\t\t=Final=: -36.33 AAGAAATCAATA T:19.0(1.02%),B:12.2(0.04%),P:1e-15\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: ATATACATATAC... (-23.670)\n",
      "\t\tRound 1: -25.32 ATATACATATAC T:28.0(1.49%),B:75.6(0.24%),P:1e-10\n",
      "\t\tRound 2: -25.32 ATATACATATAC T:28.0(1.49%),B:75.6(0.24%),P:1e-10\n",
      "\t\t=Final=: -4.69 ATATACATATAC T:7.0(0.38%),B:35.8(0.12%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: GGAAGGGAGGAA... (-19.626)\n",
      "\t\tRound 1: -21.03 GGAAGGGAGGAA T:116.0(6.04%),B:981.3(3.15%),P:1e-9\n",
      "\t\tRound 2: -21.30 GGAAGGGAGGAA T:117.0(6.09%),B:988.6(3.17%),P:1e-9\n",
      "\t\tRound 3: -21.51 GGAAGGGAGGAA T:118.0(6.15%),B:996.3(3.20%),P:1e-9\n",
      "\t\tRound 4: -21.51 GGAAGGGAGGAA T:118.0(6.15%),B:996.3(3.20%),P:1e-9\n",
      "\t\t=Final=: -7.14 GGAAGGGAGGAA T:5.0(0.27%),B:9.8(0.03%),P:1e-3\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: TAGGAAGGAAGG... (-19.773)\n",
      "\t\tRound 1: -22.50 TAGGAAGGAAGG T:117.0(6.09%),B:966.4(3.10%),P:1e-9\n",
      "\t\tRound 2: -26.06 TDGGAAGGAAGG T:126.0(6.55%),B:999.3(3.21%),P:1e-11\n",
      "\t\tRound 3: -27.81 TKGGAAGGCAGG T:129.0(6.70%),B:1005.1(3.23%),P:1e-12\n",
      "\t\tRound 4: -27.95 TKGGAAGAMAGG T:161.0(8.29%),B:1369.8(4.37%),P:1e-12\n",
      "\t\tRound 5: -27.95 TKGGAAGAMAGG T:161.0(8.29%),B:1369.8(4.37%),P:1e-12\n",
      "\t\t=Final=: -16.65 TKGGAAGAMAGG T:24.0(1.29%),B:96.0(0.31%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: AAGGAAGGAAGG... (-19.793)\n",
      "\t\tRound 1: -22.46 AAGGAAGGAAGG T:121.0(6.30%),B:1015.5(3.26%),P:1e-9\n",
      "\t\tRound 2: -25.37 AAGGAADGAAGG T:136.0(7.05%),B:1132.6(3.63%),P:1e-11\n",
      "\t\tRound 3: -25.37 AAGGAADGAAGG T:136.0(7.05%),B:1132.6(3.63%),P:1e-11\n",
      "\t\t=Final=: -7.37 AAGGAADGAAGG T:39.0(2.10%),B:354.9(1.16%),P:1e-3\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: AGGAAGGAAGGA... (-165.342)\n",
      "\t\tRound 1: -2.86 NVNAAGGAANNN T:1.0(0.05%),B:0.0(0.00%),P:1e-1\n",
      "\t\tRound 2: -5.40 DNNAAGGAANNN T:8.0(0.43%),B:31.0(0.10%),P:1e-2\n",
      "\t\tRound 3: -7.24 TAGAAGGAAGAW T:4.0(0.21%),B:2.0(0.00%),P:1e-3\n",
      "\t\tRound 4: -7.24 TAGAAGGAAGAW T:4.0(0.21%),B:2.0(0.00%),P:1e-3\n",
      "\t\t=Final=: -9.88 TAGAAGGAAGAW T:4.0(0.21%),B:2.0(0.01%),P:1e-4\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: TCTATTCTATCC... (-17.484)\n",
      "\t\tRound 1: -26.07 TCTATTCTATCC T:31.0(1.65%),B:91.0(0.30%),P:1e-11\n",
      "\t\tRound 2: -29.01 TCTATTCTATCC T:25.0(1.33%),B:46.4(0.15%),P:1e-12\n",
      "\t\tRound 3: -29.01 TCTATTCTATCC T:25.0(1.33%),B:46.4(0.15%),P:1e-12\n",
      "\t\t=Final=: -20.60 TCTATTCTATCC T:14.0(0.75%),B:19.5(0.06%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t8 of 25 Initial Sequence: TGTGTGTGTGGG... (-15.969)\n",
      "\t\tRound 1: -18.06 TGTGTGTGTGGG T:254.0(12.76%),B:2799.2(8.73%),P:1e-7\n",
      "\t\tRound 2: -19.84 TGTGTGTGTGVG T:253.0(12.71%),B:2720.3(8.49%),P:1e-8\n",
      "\t\tRound 3: -19.84 TGTGTGTGTGVG T:253.0(12.71%),B:2720.3(8.49%),P:1e-8\n",
      "\t\t=Final=: -5.39 TGTGTGTGTGVG T:15.0(0.81%),B:108.2(0.35%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t9 of 25 Initial Sequence: CCATCCATCCAT... (-15.412)\n",
      "\t\tRound 1: -17.75 CCATCCATCCAT T:35.0(1.86%),B:172.0(0.56%),P:1e-7\n",
      "\t\tRound 2: -18.66 CCATCCATCTAT T:46.0(2.44%),B:265.9(0.86%),P:1e-8\n",
      "\t\tRound 3: -18.66 CCATCCATCTAT T:46.0(2.44%),B:265.9(0.86%),P:1e-8\n",
      "\t\t=Final=: -5.83 CCATCCATCTAT T:10.0(0.54%),B:53.8(0.18%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t10 of 25 Initial Sequence: GAAATGCCCAAG... (-15.273)\n",
      "\t\tRound 1: -26.50 GAAATGCCCAAG T:24.0(1.28%),B:48.8(0.16%),P:1e-11\n",
      "\t\tRound 2: -26.50 GAAATGCCCAAG T:24.0(1.28%),B:48.8(0.16%),P:1e-11\n",
      "\t\t=Final=: -28.29 GAAATGCCCAAG T:24.0(1.29%),B:48.0(0.16%),P:1e-12\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t11 of 25 Initial Sequence: CCTGCGCAGATG... (-15.273)\n",
      "\t\tRound 1: -19.96 CCTGSGCAGATG T:11.0(0.59%),B:8.0(0.02%),P:1e-8\n",
      "\t\tRound 2: -21.93 CCTGGGCAGRTG T:11.0(0.59%),B:5.8(0.02%),P:1e-9\n",
      "\t\tRound 3: -26.46 CCTGKGAAGRTG T:26.0(1.39%),B:59.2(0.19%),P:1e-11\n",
      "\t\tRound 4: -29.94 CCTGHGAAGRTG T:22.0(1.18%),B:30.4(0.10%),P:1e-13\n",
      "\t\tRound 5: -29.94 CCTGHGAAGRTG T:22.0(1.18%),B:30.4(0.10%),P:1e-13\n",
      "\t\t=Final=: -31.48 CCTGHGAAGRTG T:22.0(1.18%),B:30.4(0.10%),P:1e-13\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t12 of 25 Initial Sequence: AAATATATATAT... (-14.482)\n",
      "\t\tRound 1: -20.20 AAATATATATAT T:372.0(18.12%),B:4296.1(13.08%),P:1e-8\n",
      "\t\tRound 2: -22.00 AAATATATATAT T:360.0(17.59%),B:4055.2(12.39%),P:1e-9\n",
      "\t\tRound 3: -22.00 AAATATATATAT T:360.0(17.59%),B:4055.2(12.39%),P:1e-9\n",
      "\t\t=Final=: -9.41 AAATATATATAT T:26.0(1.40%),B:174.9(0.57%),P:1e-4\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t13 of 25 Initial Sequence: CCATTTTCTCTT... (-14.306)\n",
      "\t\tRound 1: -14.68 CCATTTTCTCTT T:21.0(1.12%),B:77.8(0.25%),P:1e-6\n",
      "\t\tRound 2: -20.81 CCATTTTCTCTT T:20.0(1.07%),B:44.7(0.14%),P:1e-9\n",
      "\t\tRound 3: -22.13 MHATYTTCTCTT T:25.0(1.33%),B:69.5(0.22%),P:1e-9\n",
      "\t\tRound 4: -23.97 CCATTTTCTCTT T:22.0(1.18%),B:45.5(0.15%),P:1e-10\n",
      "\t\tRound 5: -23.97 CCATTTTCTCTT T:22.0(1.18%),B:45.5(0.15%),P:1e-10\n",
      "\t\t=Final=: -25.40 CCATTTTCTCTT T:22.0(1.18%),B:45.5(0.15%),P:1e-11\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t14 of 25 Initial Sequence: ATGCTGTTTGGA... (-14.306)\n",
      "\t\tRound 1: -17.17 ATGCTGTTTGGA T:7.0(0.38%),B:1.0(0.00%),P:1e-7\n",
      "\t\tRound 2: -20.03 ATTCTGATTGGA T:8.0(0.43%),B:1.0(0.00%),P:1e-8\n",
      "\t\tRound 3: -20.75 ATTCTGATTGTA T:9.0(0.48%),B:2.4(0.01%),P:1e-9\n",
      "\t\tRound 4: -20.75 ATTCTGATTGTA T:9.0(0.48%),B:2.4(0.01%),P:1e-9\n",
      "\t\t=Final=: -21.86 ATTCTGATTGTA T:9.0(0.48%),B:2.4(0.01%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t15 of 25 Initial Sequence: CATATTAAGACA... (-13.936)\n",
      "\t\tRound 1: -19.94 CATATTAAGACA T:12.0(0.64%),B:10.2(0.03%),P:1e-8\n",
      "\t\tRound 2: -20.60 CATATTAAGACA T:14.0(0.75%),B:16.7(0.05%),P:1e-8\n",
      "\t\tRound 3: -25.64 CATATWAAGACA T:37.0(1.97%),B:135.9(0.44%),P:1e-11\n",
      "\t\tRound 4: -25.64 CATATWAAGACA T:37.0(1.97%),B:135.9(0.44%),P:1e-11\n",
      "\t\t=Final=: -26.81 CATATWAAGACA T:37.0(1.99%),B:135.9(0.44%),P:1e-11\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t16 of 25 Initial Sequence: GCTCATCAGAAG... (-13.613)\n",
      "\t\tRound 1: -16.00 GCTCATCAGAAG T:9.0(0.48%),B:6.2(0.02%),P:1e-6\n",
      "\t\tRound 2: -19.40 KCTCATCAGAAG T:10.0(0.54%),B:5.6(0.02%),P:1e-8\n",
      "\t\tRound 3: -24.64 KCTSATCAGAAG T:22.0(1.18%),B:43.1(0.14%),P:1e-10\n",
      "\t\tRound 4: -24.74 GCTSATCAGAAG T:17.0(0.91%),B:20.1(0.07%),P:1e-10\n",
      "\t\tRound 5: -24.74 GCTSATCAGAAG T:17.0(0.91%),B:20.1(0.07%),P:1e-10\n",
      "\t\t=Final=: -26.87 GCTSATCAGAAG T:17.0(0.91%),B:19.2(0.06%),P:1e-11\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t17 of 25 Initial Sequence: TGATTCATCTTT... (-13.246)\n",
      "\t\tRound 1: -22.39 TGATTCATCTTT T:12.0(0.64%),B:7.0(0.02%),P:1e-9\n",
      "\t\tRound 2: -22.39 TGATTCATCTTT T:12.0(0.64%),B:7.0(0.02%),P:1e-9\n",
      "\t\t=Final=: -23.91 TGATTCATCTTT T:12.0(0.64%),B:7.0(0.02%),P:1e-10\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t18 of 25 Initial Sequence: CAGAGAGGTTTA... (-12.881)\n",
      "\t\tRound 1: -26.61 CAGAGAGGTTTA T:21.0(1.12%),B:33.2(0.11%),P:1e-11\n",
      "\t\tRound 2: -30.99 CAGAGAGGTTTW T:29.0(1.55%),B:61.7(0.20%),P:1e-13\n",
      "\t\tRound 3: -32.75 CAGAGAGGTTTW T:30.0(1.60%),B:61.7(0.20%),P:1e-14\n",
      "\t\tRound 4: -32.75 CAGAGAGGTTTW T:30.0(1.60%),B:61.7(0.20%),P:1e-14\n",
      "\t\t=Final=: -34.18 CAGAGAGGTTTW T:30.0(1.61%),B:61.7(0.20%),P:1e-14\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t19 of 25 Initial Sequence: TCCTGGAGTGGG... (-12.728)\n",
      "\t\tRound 1: -26.86 TCCTGGAGTGGG T:18.0(0.96%),B:20.3(0.07%),P:1e-11\n",
      "\t\tRound 2: -26.86 TCCTGGAGTGGG T:18.0(0.96%),B:20.3(0.07%),P:1e-11\n",
      "\t\t=Final=: -28.43 TCCTGGAGTGGG T:18.0(0.97%),B:20.3(0.07%),P:1e-12\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t20 of 25 Initial Sequence: GCTTCCCCGCAC... (-12.563)\n",
      "\t\tRound 1: -14.31 GCTTCCCCGCAC T:6.0(0.32%),B:0.9(0.00%),P:1e-6\n",
      "\t\tRound 2: -20.50 TCTTCCCCMCAC T:40.0(2.13%),B:193.7(0.63%),P:1e-8\n",
      "\t\tRound 3: -20.50 TCTTCCCCMCAC T:40.0(2.13%),B:193.7(0.63%),P:1e-8\n",
      "\t\t=Final=: -19.55 TCTTCCCCMCAC T:37.0(1.99%),B:182.9(0.60%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t21 of 25 Initial Sequence: ACCATCCCTTGG... (-12.728)\n",
      "\t\tRound 1: -20.03 ACCATCCCTTGG T:8.0(0.43%),B:0.8(0.00%),P:1e-8\n",
      "\t\tRound 2: -20.75 ACCCTCCTTTGG T:9.0(0.48%),B:2.2(0.01%),P:1e-9\n",
      "\t\tRound 3: -24.54 RCMMTYCYTTGS T:11.0(0.59%),B:3.2(0.01%),P:1e-10\n",
      "\t\tRound 4: -24.54 RCMMTYCYTTGS T:11.0(0.59%),B:3.2(0.01%),P:1e-10\n",
      "\t\t=Final=: -25.76 RCMMTYCYTTGS T:11.0(0.59%),B:3.2(0.01%),P:1e-11\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t22 of 25 Initial Sequence: GCAAGCAGGGTA... (-12.563)\n",
      "\t\tRound 1: -20.60 GCAAGCAGGGTA T:15.0(0.80%),B:20.2(0.07%),P:1e-8\n",
      "\t\tRound 2: -20.60 GCAAGCAGGGTA T:15.0(0.80%),B:20.2(0.07%),P:1e-8\n",
      "\t\t=Final=: -22.15 GCAAGCAGGGTA T:15.0(0.81%),B:20.2(0.07%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t23 of 25 Initial Sequence: ACTGCTGCAGAG... (-12.233)\n",
      "\t\tRound 1: -20.59 DNSGCBGCSSNB T:18.0(0.96%),B:34.8(0.11%),P:1e-8\n",
      "\t\tRound 2: -27.97 GSCGCKGCCSYG T:30.0(1.60%),B:77.2(0.25%),P:1e-12\n",
      "\t\tRound 3: -28.89 GSSGCKGCCSCG T:21.0(1.12%),B:28.3(0.09%),P:1e-12\n",
      "\t\tRound 4: -28.89 GSSGCKGCCSCG T:21.0(1.12%),B:28.3(0.09%),P:1e-12\n",
      "\t\t=Final=: -10.72 GSSGCKGCCSCG T:11.0(0.59%),B:32.2(0.11%),P:1e-4\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t24 of 25 Initial Sequence: ACTTCTAATTAA... (-11.774)\n",
      "\t\tRound 1: -20.03 ACTTCTAATTAA T:8.0(0.43%),B:0.0(0.00%),P:1e-8\n",
      "\t\tRound 2: -20.03 ACTTCTAATTAA T:8.0(0.43%),B:0.0(0.00%),P:1e-8\n",
      "\t\t=Final=: -22.90 ACTTCTAATTAA T:8.0(0.43%),B:0.0(0.00%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t25 of 25 Initial Sequence: CCTCATTTCCTA... (-11.671)\n",
      "\t\tRound 1: -18.02 CCTCATTTCCTA T:12.0(0.64%),B:13.3(0.04%),P:1e-7\n",
      "\t\tRound 2: -18.46 CCTCATTTCCTA T:17.0(0.91%),B:35.2(0.11%),P:1e-8\n",
      "\t\tRound 3: -19.59 CCTCAKTTCCTA T:13.0(0.70%),B:14.9(0.05%),P:1e-8\n",
      "\t\tRound 4: -20.12 CCTCAKTTCCTA T:15.0(0.80%),B:21.3(0.07%),P:1e-8\n",
      "\t\tRound 5: -23.13 CCWCAGTTCCTA T:11.0(0.59%),B:4.7(0.01%),P:1e-10\n",
      "\t\tRound 6: -23.13 CCWCAGTTCCTA T:11.0(0.59%),B:4.7(0.01%),P:1e-10\n",
      "\t\t=Final=: -24.49 CCWCAGTTCCTA T:11.0(0.59%),B:4.7(0.02%),P:1e-10\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t32939 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing hypergeometric scoring\n",
      "\tChecking enrichment of 25 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: motifResults/HOMER_DENOVO_treatedF03_normalFVB.filt.bed_vs_GATK.dbSNP.bed/homerMotifs.motifs12\n",
      "\n",
      "\t(Motifs in homer2 format)\n",
      "\tDetermining similar motifs... 57 reduced to 37 motifs\n",
      "\tOutputing HTML and sequence logos for motif comparison...\n",
      "\tChecking de novo motifs against known motifs...\n",
      "\tFormatting HTML page...\n",
      "\t\t1 of 37 (1e-15) similar to ONECUT3/MA0757.1/Jaspar(0.851)\n",
      "\t\t2 of 37 (1e-14) similar to TUT1(RRM,Znf)/Homo_sapiens-RNCMPT00075-PBM/HughesRNA(0.702)\n",
      "\t\t3 of 37 (1e-14) similar to SRSF10(RRM)/Homo_sapiens-RNCMPT00088-PBM/HughesRNA(0.806)\n",
      "\t\t4 of 37 (1e-13) similar to Rbpj1(?)/Panc1-Rbpj1-ChIP-Seq(GSE47459)/Homer(0.674)\n",
      "\t\t5 of 37 (1e-13) similar to PB0022.1_Gata5_1/Jaspar(0.699)\n",
      "\t\t6 of 37 (1e-12) similar to MAFK/MA0496.1/Jaspar(0.678)\n",
      "\t\t7 of 37 (1e-12) similar to PB0114.1_Egr1_2/Jaspar(0.665)\n",
      "\t\t8 of 37 (1e-12) similar to PB0133.1_Hic1_2/Jaspar(0.715)\n",
      "\t\t9 of 37 (1e-11) similar to REF2(RRM)/Drosophila_melanogaster-RNCMPT00059-PBM/HughesRNA(0.705)\n",
      "\t\t10 of 37 (1e-11) similar to SGR5(C2H2)/colamp-SGR5-DAP-Seq(GSE60143)/Homer(0.672)\n",
      "\t\t11 of 37 (1e-11) similar to CG5213(RRM)/Drosophila_melanogaster-RNCMPT00010-PBM/HughesRNA(0.610)\n",
      "\t\t12 of 37 (1e-11) similar to SRSF10(RRM)/Homo_sapiens-RNCMPT00019-PBM/HughesRNA(0.763)\n",
      "\t\t13 of 37 (1e-10) similar to Chop(bZIP)/MEF-Chop-ChIP-Seq(GSE35681)/Homer(0.726)\n",
      "\t\t14 of 37 (1e-10) similar to MET4(MacIsaac)/Yeast(0.733)\n",
      "\t\t15 of 37 (1e-10) similar to PB0201.1_Zfp281_2/Jaspar(0.644)\n",
      "\t\t16 of 37 (1e-10) similar to MBNL1(Znf)/Homo_sapiens-RNCMPT00038-PBM/HughesRNA(0.678)\n",
      "\t\t17 of 37 (1e-10) similar to JUN(var.2)/MA0489.1/Jaspar(0.758)\n",
      "\t\t18 of 37 (1e-10) similar to SRS7(SRS)/colamp-SRS7-DAP-Seq(GSE60143)/Homer(0.782)\n",
      "\t\t19 of 37 (1e-9) similar to Al/dmmpmm(Noyes_hd)/fly(0.783)\n",
      "\t\t20 of 37 (1e-9) similar to JUND(var.2)/MA0492.1/Jaspar(0.749)\n",
      "\t\t21 of 37 (1e-9) similar to Dux/MA0611.1/Jaspar(0.698)\n",
      "\t\t22 of 37 (1e-9) similar to RPN4/RPN4_H2O2Lo/[](Harbison)/Yeast(0.853)\n",
      "\t\t23 of 37 (1e-9) similar to KHDRBS1(KH)/Mus_musculus-RNCMPT00062-PBM/HughesRNA(0.669)\n",
      "\t\t24 of 37 (1e-9) similar to PUM(PUF)/Drosophila_melanogaster-RNCMPT00101-PBM/HughesRNA(0.673)\n",
      "\t\t25 of 37 (1e-8) similar to GCR2/MA0305.1/Jaspar(0.714)\n",
      "\t\t26 of 37 (1e-8) similar to hb/dmmpmm(Bergman)/fly(0.658)\n",
      "\t\t27 of 37 (1e-8) similar to RO3G_00049(RRM)/Rhizopus_oryzae-RNCMPT00205-PBM/HughesRNA(0.807)\n",
      "\t\t28 of 37 (1e-8) similar to GSC(Homeobox)/FrogEmbryos-GSC-ChIP-Seq(DRA000576)/Homer(0.701)\n",
      "\t\t29 of 37 (1e-7) similar to ARG80(MacIsaac)/Yeast(0.739)\n",
      "\t\t30 of 37 (1e-7) similar to RIN(RRM)/Drosophila_melanogaster-RNCMPT00138-PBM/HughesRNA(0.844)\n",
      "\t\t31 of 37 (1e-6) similar to GATA14(C2C2gata)/col-GATA14-DAP-Seq(GSE60143)/Homer(0.840)\n",
      "\t\t32 of 37 (1e-5) similar to ERF73(AP2EREBP)/col-ERF73-DAP-Seq(GSE60143)/Homer(0.837)\n",
      "\t\t33 of 37 (1e-5) similar to PB0100.1_Zfp740_1/Jaspar(0.896)\n",
      "\t\t34 of 37 (1e-4) similar to Nur77(NR)/K562-NR4A1-ChIP-Seq(GSE31363)/Homer(0.849)\n",
      "\t\t35 of 37 (1e-4) similar to SeqBias: TA-repeat(0.814)\n",
      "\t\t36 of 37 (1e-3) similar to PH0084.1_Irx3_2/Jaspar(0.766)\n",
      "\t\t37 of 37 (1e-2) similar to SeqBias: CA-repeat(0.915)\n",
      "\tJob finished\n",
      "\n",
      "treatedFVB_normalF03.filt.bed\n",
      "\n",
      "Selected Options:\n",
      "\tInput file = treatedFVB_normalF03.filt.100bp.fa\n",
      "\tPromoter Set = fasta\n",
      "\tOutput Directory = motifResults/HOMER_DENOVO_treatedFVB_normalF03.filt.bed_vs_GATK.dbSNP.bed\n",
      "\tWill use FASTA files for motif finding\n",
      "\t\tTarget Sequences = treatedFVB_normalF03.filt.100bp.fa\n",
      "\t\tBackground Sequences = GATK.dbSNP.100bp.fa\n",
      "\tWill not search for known motifs\n",
      "\tUsing custom gene IDs for GO analysis\n",
      "\tParsing FASTA format files...\n",
      "\tFound 295 sequences\n",
      "\tFound 31078 sequences\n",
      "\n",
      "\tProgress: Step4 - removing redundant promoters\n",
      "\n",
      "\tProgress: Step5 - adjusting background sequences for GC/CpG content...\n",
      "\n",
      "\tSequences processed:\n",
      "\t\tAuto detected maximum sequence length of 101 bp\n",
      "\t\t31373 total\n",
      "\n",
      "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
      "\tFreq\tBin\tCount\n",
      "\t0.2\t0\t324\n",
      "\t0.25\t1\t787\n",
      "\t0.3\t2\t2373\n",
      "\t0.35\t3\t4458\n",
      "\t0.4\t4\t6204\n",
      "\t0.45\t5\t6033\n",
      "\t0.5\t6\t5638\n",
      "\t0.6\t7\t4904\n",
      "\t0.7\t8\t566\n",
      "\t0.8\t9\t82\n",
      "\t10\t10\t4\n",
      "\tBin\t# Targets\t# Background\tBackground Weight\n",
      "\t0\t2\t322\t0.654\n",
      "\t1\t8\t779\t1.082\n",
      "\t2\t21\t2352\t0.941\n",
      "\t3\t45\t4413\t1.074\n",
      "\t4\t44\t6160\t0.752\n",
      "\t5\t54\t5979\t0.951\n",
      "\t6\t68\t5570\t1.286\n",
      "\t7\t42\t4862\t0.910\n",
      "\t8\t8\t558\t1.510\n",
      "\t9\t2\t80\t2.634\n",
      "\t10\t1\t3\t35.116\n",
      "\n",
      "\tNormalizing lower order oligos using homer2\n",
      "\n",
      "\tReading input files...\n",
      "\t31373 total sequences read\n",
      "\tAutonormalization: 1-mers (4 total)\n",
      "\t\tA\t28.84%\t28.89%\t0.998\n",
      "\t\tC\t21.16%\t21.11%\t1.002\n",
      "\t\tG\t21.16%\t21.11%\t1.002\n",
      "\t\tT\t28.84%\t28.89%\t0.998\n",
      "\tAutonormalization: 2-mers (16 total)\n",
      "\t\tAA\t8.92%\t8.68%\t1.028\n",
      "\t\tCA\t7.27%\t7.45%\t0.976\n",
      "\t\tGA\t6.36%\t6.43%\t0.990\n",
      "\t\tTA\t6.27%\t6.32%\t0.991\n",
      "\t\tAC\t5.09%\t5.34%\t0.952\n",
      "\t\tCC\t5.45%\t5.28%\t1.032\n",
      "\t\tGC\t4.28%\t4.07%\t1.050\n",
      "\t\tTC\t6.36%\t6.43%\t0.990\n",
      "\t\tAG\t7.41%\t7.42%\t0.998\n",
      "\t\tCG\t1.03%\t0.96%\t1.075\n",
      "\t\tGG\t5.45%\t5.28%\t1.032\n",
      "\t\tTG\t7.27%\t7.45%\t0.976\n",
      "\t\tAT\t7.44%\t7.45%\t0.998\n",
      "\t\tCT\t7.41%\t7.42%\t0.998\n",
      "\t\tGT\t5.09%\t5.34%\t0.952\n",
      "\t\tTT\t8.92%\t8.68%\t1.028\n",
      "\tAutonormalization: 3-mers (64 total)\n",
      "\tNormalization weights can be found in file: motifResults/HOMER_DENOVO_treatedFVB_normalF03.filt.bed_vs_GATK.dbSNP.bed/seq.autonorm.tsv\n",
      "\tConverging on autonormalization solution:\n",
      "\t...............................................................................\n",
      "\tFinal normalization:\tAutonormalization: 1-mers (4 total)\n",
      "\t\tA\t28.84%\t28.84%\t1.000\n",
      "\t\tC\t21.16%\t21.16%\t1.000\n",
      "\t\tG\t21.16%\t21.16%\t1.000\n",
      "\t\tT\t28.84%\t28.84%\t1.000\n",
      "\tAutonormalization: 2-mers (16 total)\n",
      "\t\tAA\t8.92%\t8.91%\t1.001\n",
      "\t\tCA\t7.27%\t7.28%\t1.000\n",
      "\t\tGA\t6.36%\t6.37%\t0.999\n",
      "\t\tTA\t6.27%\t6.28%\t0.998\n",
      "\t\tAC\t5.09%\t5.09%\t1.000\n",
      "\t\tCC\t5.45%\t5.44%\t1.001\n",
      "\t\tGC\t4.28%\t4.27%\t1.002\n",
      "\t\tTC\t6.36%\t6.37%\t0.999\n",
      "\t\tAG\t7.41%\t7.41%\t0.999\n",
      "\t\tCG\t1.03%\t1.02%\t1.007\n",
      "\t\tGG\t5.45%\t5.44%\t1.001\n",
      "\t\tTG\t7.27%\t7.28%\t1.000\n",
      "\t\tAT\t7.44%\t7.44%\t1.000\n",
      "\t\tCT\t7.41%\t7.41%\t0.999\n",
      "\t\tGT\t5.09%\t5.09%\t1.000\n",
      "\t\tTT\t8.92%\t8.91%\t1.001\n",
      "\tAutonormalization: 3-mers (64 total)\n",
      "\n",
      "\tProgress: Step6 - Gene Ontology Enrichment Analysis\n",
      "\tSkipping...\n",
      "\n",
      "\tProgress: Step7 - Known motif enrichment\n",
      "\tSkipping...\n",
      "\n",
      "\tProgress: Step8 - De novo motif finding (HOMER)\n",
      "\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 32427\n",
      "\tAutoadjustment for sequence coverage in background: 1.00x\n",
      "\n",
      "\tOligos: 32427 of 34497 max\n",
      "\tTree  : 66272 of 172485 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 1 mismatches...\n",
      "\n",
      "\tScreening oligos 32427 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t54.44% skipped, 45.56% checked (14775 of 32427), of those checked:\n",
      "\t\t54.44% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 32427 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t54.44% skipped, 45.56% checked (14775 of 32427), of those checked:\n",
      "\t\t0.00% not in target, 22.55% increased p-value, 30.49% high p-value\n",
      "\tReading input files...\n",
      "\t31373 total sequences read\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: TCTATTCT... (-17.939)\n",
      "\t\tRound 1: -21.74 TCTATTCT T:33.0(10.60%),B:857.2(2.73%),P:1e-9\n",
      "\t\tRound 2: -21.74 TCTATTCT T:33.0(10.60%),B:857.2(2.73%),P:1e-9\n",
      "\t\t=Final=: -3.95 TCTATTCT T:13.0(4.41%),B:702.6(2.27%),P:1e-1\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: AAGGAAGG... (-15.907)\n",
      "\t\tRound 1: -18.05 AAGGAAGG T:58.0(17.88%),B:2437.5(7.58%),P:1e-7\n",
      "\t\tRound 2: -21.10 AAGTAAGG T:82.0(24.30%),B:3703.6(11.29%),P:1e-9\n",
      "\t\tRound 3: -21.10 AAGTAAGG T:82.0(24.30%),B:3703.6(11.29%),P:1e-9\n",
      "\t\t=Final=: -10.73 AAGTAAGG T:38.0(12.88%),B:1923.2(6.22%),P:1e-4\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: TATATATA... (-11.586)\n",
      "\t\tRound 1: -12.01 TATATATA T:64.0(19.53%),B:3446.0(10.55%),P:1e-5\n",
      "\t\tRound 2: -14.45 TCTATATA T:125.0(34.59%),B:7661.8(21.95%),P:1e-6\n",
      "\t\tRound 3: -14.45 TCTATATA T:125.0(34.59%),B:7661.8(21.95%),P:1e-6\n",
      "\t\t=Final=: -3.53 TCTATATA T:30.0(10.17%),B:2178.2(7.04%),P:1e-1\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: AGAGGCTA... (-9.406)\n",
      "\t\tRound 1: -12.82 AGAGGCTA T:11.0(3.67%),B:148.3(0.48%),P:1e-5\n",
      "\t\tRound 2: -12.82 AGAGGCTA T:11.0(3.67%),B:148.3(0.48%),P:1e-5\n",
      "\t\t=Final=: -15.06 AGAGGCTA T:11.0(3.73%),B:144.5(0.47%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: GTTGTGCC... (-8.414)\n",
      "\t\tRound 1: -13.90 GTTGTGCC T:75.0(22.48%),B:4028.6(12.22%),P:1e-6\n",
      "\t\tRound 2: -13.90 GTTGTGCC T:75.0(22.48%),B:4028.6(12.22%),P:1e-6\n",
      "\t\t=Final=: -12.15 GTTGTGCC T:63.0(21.36%),B:3730.1(12.06%),P:1e-5\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\tRemaining seeds don't look promising (After initial 5 motifs, logp -7.257 > -8.470)\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t31373 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing hypergeometric scoring\n",
      "\tChecking enrichment of 5 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: motifResults/HOMER_DENOVO_treatedFVB_normalF03.filt.bed_vs_GATK.dbSNP.bed/homerMotifs.motifs8\n",
      "\n",
      "\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 355032\n",
      "\tAutoadjustment for sequence coverage in background: 1.00x\n",
      "\n",
      "\tOligos: 355032 of 551887 max\n",
      "\tTree  : 766288 of 2759435 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 1 mismatches...\n",
      "\n",
      "\tScreening oligos 355032 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t93.28% skipped, 6.72% checked (23863 of 355032), of those checked:\n",
      "\t\t93.28% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 355032 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t93.28% skipped, 6.72% checked (23863 of 355032), of those checked:\n",
      "\t\t0.00% not in target, 6.15% increased p-value, 0.65% high p-value\n",
      "\tReading input files...\n",
      "\t31373 total sequences read\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: GAATAGAATA... (-28.388)\n",
      "\t\tRound 1: -30.66 GAATAGAATA T:14.0(4.64%),B:54.6(0.17%),P:1e-13\n",
      "\t\tRound 2: -33.26 TAATAGAMTA T:16.0(5.29%),B:72.8(0.23%),P:1e-14\n",
      "\t\tRound 3: -33.26 TAATAGAMTA T:16.0(5.29%),B:72.8(0.23%),P:1e-14\n",
      "\t\t=Final=: -9.26 TAATAGAMTA T:3.0(1.02%),B:7.8(0.03%),P:1e-4\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: AATAGAATAG... (-26.982)\n",
      "\t\tRound 1: -34.71 AATAGAATAG T:50.0(15.61%),B:1186.1(3.76%),P:1e-15\n",
      "\t\tRound 2: -34.71 AATAGAATAG T:50.0(15.61%),B:1186.1(3.76%),P:1e-15\n",
      "\t\t=Final=: -9.40 AATAGAATAG T:15.0(5.08%),B:479.3(1.55%),P:1e-4\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: CCCTTCCTTC... (-16.866)\n",
      "\t\tRound 1: -23.94 CCCTTCCTTC T:62.0(18.98%),B:2313.5(7.21%),P:1e-10\n",
      "\t\tRound 2: -23.94 CCCTTCCTTC T:62.0(18.98%),B:2313.5(7.21%),P:1e-10\n",
      "\t\t=Final=: -13.01 CCCTTCCTTC T:26.0(8.81%),B:946.2(3.06%),P:1e-5\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: ATATATATAT... (-14.561)\n",
      "\t\tRound 1: -16.60 ATATATATAT T:61.0(18.71%),B:2793.7(8.64%),P:1e-7\n",
      "\t\tRound 2: -19.20 AAAWATATAT T:67.0(20.35%),B:2966.9(9.15%),P:1e-8\n",
      "\t\tRound 3: -19.20 AAAWATATAT T:67.0(20.35%),B:2966.9(9.15%),P:1e-8\n",
      "\t\t=Final=: -5.69 AAAWATATAT T:4.0(1.36%),B:61.7(0.20%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: TATATAGATA... (-10.700)\n",
      "\t\tRound 1: -14.24 TATATAGATA T:105.0(29.99%),B:6152.5(18.05%),P:1e-6\n",
      "\t\tRound 2: -14.94 TATATAGAAA T:113.0(31.87%),B:6683.1(19.44%),P:1e-6\n",
      "\t\tRound 3: -14.94 TATATAGAAA T:113.0(31.87%),B:6683.1(19.44%),P:1e-6\n",
      "\t\t=Final=: -9.89 TATATAGAAA T:7.0(2.37%),B:94.3(0.31%),P:1e-4\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: GCTGCCCCTC... (-9.963)\n",
      "\t\tRound 1: -18.48 GCTGCCCCTC T:24.0(7.83%),B:563.7(1.80%),P:1e-8\n",
      "\t\tRound 2: -18.80 GCTGCCCCTC T:25.0(8.14%),B:601.7(1.93%),P:1e-8\n",
      "\t\tRound 3: -18.80 GCTGCCCCTC T:25.0(8.14%),B:601.7(1.93%),P:1e-8\n",
      "\t\t=Final=: -20.03 GCTGCCCCTC T:25.0(8.47%),B:604.7(1.96%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: AGGAACCTGC... (-9.735)\n",
      "\t\tRound 1: -25.26 AGGAACCTGC T:24.0(7.83%),B:389.1(1.25%),P:1e-10\n",
      "\t\tRound 2: -25.26 AGGAACCTGC T:24.0(7.83%),B:389.1(1.25%),P:1e-10\n",
      "\t\t=Final=: -23.78 AGGAACCTGC T:22.0(7.46%),B:378.1(1.22%),P:1e-10\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t8 of 25 Initial Sequence: ATCAGAATGG... (-9.606)\n",
      "\t\tRound 1: -11.53 ATCAGAATGG T:5.0(1.68%),B:12.5(0.04%),P:1e-5\n",
      "\t\tRound 2: -13.96 AGCAGAATGG T:25.0(8.14%),B:792.4(2.53%),P:1e-6\n",
      "\t\tRound 3: -13.96 AGCAGAATGG T:25.0(8.14%),B:792.4(2.53%),P:1e-6\n",
      "\t\t=Final=: -15.32 AGCAGAATGG T:25.0(8.47%),B:777.0(2.51%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t9 of 25 Initial Sequence: ATGATGGCTT... (-9.585)\n",
      "\t\tRound 1: -16.54 ATGATGGCTT T:12.0(3.99%),B:124.5(0.40%),P:1e-7\n",
      "\t\tRound 2: -16.54 ATGATGGCTT T:12.0(3.99%),B:124.5(0.40%),P:1e-7\n",
      "\t\t=Final=: -18.74 ATGATGGCTT T:12.0(4.07%),B:124.5(0.40%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t10 of 25 Initial Sequence: CACCATCAGT... (-9.326)\n",
      "\t\tRound 1: -9.61 CACCATCAGT T:4.0(1.35%),B:8.0(0.02%),P:1e-4\n",
      "\t\tRound 2: -9.61 CACCATCAGT T:4.0(1.35%),B:8.0(0.02%),P:1e-4\n",
      "\t\t=Final=: -13.99 CACCATCAGT T:3.0(1.02%),B:0.8(0.00%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t11 of 25 Initial Sequence: TGATGATGAT... (-9.028)\n",
      "\t\tRound 1: -13.42 TGATGATGAT T:9.0(3.01%),B:78.1(0.25%),P:1e-5\n",
      "\t\tRound 2: -13.42 TGATGATGAT T:9.0(3.01%),B:78.1(0.25%),P:1e-5\n",
      "\t\t=Final=: -2.72 TGATGATGAT T:60.0(20.34%),B:5201.7(16.82%),P:1e-1\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t12 of 25 Initial Sequence: GATGATGATG... (-23.341)\n",
      "\t\tRound 1: -4.92 GATGATGATG T:4.0(1.35%),B:40.3(0.13%),P:1e-2\n",
      "\t\tRound 2: -4.92 GATGATGATG T:4.0(1.35%),B:40.3(0.13%),P:1e-2\n",
      "\t\t=Final=: -4.99 GATGATGATG T:3.0(1.02%),B:38.6(0.12%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\tRemaining seeds don't look promising (After initial 5 motifs, logp -8.079 > -8.470)\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t31373 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing hypergeometric scoring\n",
      "\tChecking enrichment of 12 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: motifResults/HOMER_DENOVO_treatedFVB_normalF03.filt.bed_vs_GATK.dbSNP.bed/homerMotifs.motifs10\n",
      "\n",
      "\n",
      "\t-blen automatically set to 2\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 1214451\n",
      "\tAutoadjustment for sequence coverage in background: 1.00x\n",
      "\n",
      "\tOligos: 1214451 of 2823570 max\n",
      "\tTree  : 3134980 of 14117850 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 1 mismatches...\n",
      "\n",
      "\tScreening oligos 1214451 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t97.97% skipped, 2.03% checked (24690 of 1214451), of those checked:\n",
      "\t\t97.97% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 1214451 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t97.97% skipped, 2.03% checked (24690 of 1214451), of those checked:\n",
      "\t\t0.00% not in target, 1.96% increased p-value, 0.08% high p-value\n",
      "\tReading input files...\n",
      "\t31373 total sequences read\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: TCGAATAGAATA... (-29.717)\n",
      "\t\tRound 1: -34.02 TAGAATAAAATA T:14.0(4.64%),B:40.1(0.13%),P:1e-14\n",
      "\t\tRound 2: -34.02 TAGAATAAAATA T:14.0(4.64%),B:40.1(0.13%),P:1e-14\n",
      "\t\t=Final=: -5.04 TAGAATAAAATA T:2.0(0.68%),B:11.9(0.04%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: CTTTTCTCTTCT... (-19.489)\n",
      "\t\tRound 1: -22.49 CTYTTCTYTTCT T:23.0(7.51%),B:409.5(1.31%),P:1e-9\n",
      "\t\tRound 2: -24.12 CTCTTGTTTTCT T:25.0(8.14%),B:454.3(1.46%),P:1e-10\n",
      "\t\tRound 3: -24.12 CTCTTGTTTTCT T:25.0(8.14%),B:454.3(1.46%),P:1e-10\n",
      "\t\t=Final=: -4.77 CTCTTGTTTTCT T:5.0(1.69%),B:127.1(0.41%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: TTCCTTCCTCCC... (-19.018)\n",
      "\t\tRound 1: -24.20 TTCCTYCCTCCC T:116.0(32.56%),B:5659.3(16.73%),P:1e-10\n",
      "\t\tRound 2: -31.12 BBCCTCCYTCCC T:94.0(27.33%),B:3624.0(11.06%),P:1e-13\n",
      "\t\tRound 3: -32.46 TTCCTCCYTCCC T:95.0(27.57%),B:3607.8(11.01%),P:1e-14\n",
      "\t\tRound 4: -32.46 TTCCTCCYTCCC T:95.0(27.57%),B:3607.8(11.01%),P:1e-14\n",
      "\t\t=Final=: -3.62 TTCCTCCYTCCC T:2.0(0.68%),B:25.7(0.08%),P:1e-1\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: GATGATGATTAT... (-16.999)\n",
      "\t\tRound 1: -22.49 GATGATGACTAT T:11.0(3.67%),B:49.4(0.16%),P:1e-9\n",
      "\t\tRound 2: -22.49 GATGATGACTAT T:11.0(3.67%),B:49.4(0.16%),P:1e-9\n",
      "\t\t=Final=: -13.99 GATGATGACTAT T:3.0(1.02%),B:0.7(0.00%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: TTCTTTCCTTCC... (-16.258)\n",
      "\t\tRound 1: -19.68 TTCTTTCCTTCC T:102.0(29.27%),B:5208.9(15.50%),P:1e-8\n",
      "\t\tRound 2: -24.48 TTCTTTCCTTCC T:60.0(18.43%),B:2139.6(6.69%),P:1e-10\n",
      "\t\tRound 3: -24.48 TTCTTTCCTTCC T:60.0(18.43%),B:2139.6(6.69%),P:1e-10\n",
      "\t\t=Final=: -9.90 TTCTTTCCTTCC T:18.0(6.10%),B:628.6(2.03%),P:1e-4\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: ATATATATATAT... (-15.211)\n",
      "\t\tRound 1: -19.53 MTATATATATAT T:59.0(18.15%),B:2399.2(7.47%),P:1e-8\n",
      "\t\tRound 2: -21.44 MTMWATATATAT T:65.0(19.81%),B:2632.2(8.16%),P:1e-9\n",
      "\t\tRound 3: -21.44 MTMWATATATAT T:65.0(19.81%),B:2632.2(8.16%),P:1e-9\n",
      "\t\t=Final=: -3.16 MTMWATATATAT T:7.0(2.37%),B:331.4(1.07%),P:1e-1\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: GATCCTCCAATA... (-13.545)\n",
      "\t\tRound 1: -18.70 GATCCTCCAAYA T:12.0(3.99%),B:99.5(0.32%),P:1e-8\n",
      "\t\tRound 2: -24.05 GATCCTMCAAHA T:12.0(3.99%),B:57.8(0.18%),P:1e-10\n",
      "\t\tRound 3: -24.05 GATCCTMCAAHA T:12.0(3.99%),B:57.8(0.18%),P:1e-10\n",
      "\t\t=Final=: -18.21 GATCCTMCAAHA T:9.0(3.05%),B:57.8(0.19%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t8 of 25 Initial Sequence: CAATCAGAATGG... (-12.615)\n",
      "\t\tRound 1: -12.62 CAATCAGAATGG T:4.0(1.35%),B:1.9(0.00%),P:1e-5\n",
      "\t\tRound 2: -12.62 CAATCAGAATGG T:4.0(1.35%),B:1.9(0.00%),P:1e-5\n",
      "\t\t=Final=: -17.06 CAATCAGAATGG T:4.0(1.36%),B:1.9(0.01%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t9 of 25 Initial Sequence: GGCTGTTTGTGG... (-10.004)\n",
      "\t\tRound 1: -14.70 GRCTGTTYGTGG T:6.0(2.02%),B:13.6(0.04%),P:1e-6\n",
      "\t\tRound 2: -14.70 GRCTGTTYGTGG T:6.0(2.02%),B:13.6(0.04%),P:1e-6\n",
      "\t\t=Final=: -17.92 GRCTGTTYGTGG T:6.0(2.03%),B:13.6(0.04%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t10 of 25 Initial Sequence: TTTGAAATTAAT... (-9.326)\n",
      "\t\tRound 1: -13.36 TTTAAAATTAAT T:11.0(3.67%),B:139.3(0.45%),P:1e-5\n",
      "\t\tRound 2: -15.96 TTTAAAATKWAT T:10.0(3.34%),B:77.5(0.25%),P:1e-6\n",
      "\t\tRound 3: -15.96 TTTAAAATKWAT T:10.0(3.34%),B:77.5(0.25%),P:1e-6\n",
      "\t\t=Final=: -18.39 TTTAAAATKWAT T:10.0(3.39%),B:77.5(0.25%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t11 of 25 Initial Sequence: ACACCGGTTCCG... (-9.326)\n",
      "\t\tRound 1: -15.97 ACSCYGGTTCCK T:5.0(1.68%),B:3.5(0.01%),P:1e-6\n",
      "\t\tRound 2: -17.06 ACVCCGGTTCCG T:5.0(1.68%),B:1.4(0.00%),P:1e-7\n",
      "\t\tRound 3: -18.67 ACACCGGTTCCG T:5.0(1.68%),B:0.0(0.00%),P:1e-8\n",
      "\t\tRound 4: -18.67 ACACCGGTTCCG T:5.0(1.68%),B:0.0(0.00%),P:1e-8\n",
      "\t\t=Final=: -4.66 ACACCGGTTCCG T:1.0(0.34%),B:0.0(0.00%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t12 of 25 Initial Sequence: AAAATGAGAGGA... (-8.642)\n",
      "\t\tRound 1: -14.45 AAAATGAGAGGA T:5.0(1.68%),B:5.7(0.02%),P:1e-6\n",
      "\t\tRound 2: -14.45 AAAATGAGAGGA T:5.0(1.68%),B:5.7(0.02%),P:1e-6\n",
      "\t\t=Final=: -17.85 AAAATGAGAGGA T:5.0(1.69%),B:5.7(0.02%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t13 of 25 Initial Sequence: GGTCAAATGCTT... (-8.511)\n",
      "\t\tRound 1: -13.47 GGTCAAAKGCTT T:8.0(2.68%),B:53.8(0.17%),P:1e-5\n",
      "\t\tRound 2: -15.51 GGTCAAAGGCTT T:9.0(3.01%),B:58.2(0.19%),P:1e-6\n",
      "\t\tRound 3: -15.51 GGTCAAAGGCTT T:9.0(3.01%),B:58.2(0.19%),P:1e-6\n",
      "\t\t=Final=: -18.08 GGTCAAAGGCTT T:9.0(3.05%),B:58.2(0.19%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\tRemaining seeds don't look promising (After initial 5 motifs, logp -8.333 > -8.470)\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t31373 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing hypergeometric scoring\n",
      "\tChecking enrichment of 13 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: motifResults/HOMER_DENOVO_treatedFVB_normalF03.filt.bed_vs_GATK.dbSNP.bed/homerMotifs.motifs12\n",
      "\n",
      "\t(Motifs in homer2 format)\n",
      "\tDetermining similar motifs... 30 reduced to 17 motifs\n",
      "\tOutputing HTML and sequence logos for motif comparison...\n",
      "\tChecking de novo motifs against known motifs...\n",
      "\tFormatting HTML page...\n",
      "\t\t1 of 17 (1e-10) similar to TEIL(AP2/EREBP)/Nicotiana tabacum/AthaMap(0.696)\n",
      "\t\t2 of 17 (1e-9) similar to RGM1/MA0366.1/Jaspar(0.684)\n",
      "\t\t3 of 17 (1e-8) similar to PB0137.1_Irf3_2/Jaspar(0.636)\n",
      "\t\t4 of 17 (1e-8) similar to HOXA2(Homeobox)/mES-Hoxa2-ChIP-Seq(Donaldson_et_al.)/Homer(0.698)\n",
      "\t\t5 of 17 (1e-7) similar to SF2(RRM)/Drosophila_melanogaster-RNCMPT00066-PBM/HughesRNA(0.698)\n",
      "\t\t6 of 17 (1e-7) similar to Unknown6/Drosophila-Promoters/Homer(0.739)\n",
      "\t\t7 of 17 (1e-7) similar to Nr2e3/MA0164.1/Jaspar(0.647)\n",
      "\t\t8 of 17 (1e-7) similar to Tv_0258(RRM)/Trichomonas_vaginalis-RNCMPT00258-PBM/HughesRNA(0.673)\n",
      "\t\t9 of 17 (1e-7) similar to tll/dmmpmm(Papatsenko)/fly(0.692)\n",
      "\t\t10 of 17 (1e-6) similar to CRZ1(MacIsaac)/Yeast(0.683)\n",
      "\t\t11 of 17 (1e-6) similar to PH0017.1_Cux1_2/Jaspar(0.688)\n",
      "\t\t12 of 17 (1e-5) similar to RBM5(Znf)/Homo_sapiens-RNCMPT00055-PBM/HughesRNA(0.759)\n",
      "\t\t13 of 17 (1e-5) similar to UNC-75(RRM)/Caenorhabditis_elegans-RNCMPT00081-PBM/HughesRNA(0.815)\n",
      "\t\t14 of 17 (1e-4) similar to SeqBias: TA-repeat(0.785)\n",
      "\t\t15 of 17 (1e-4) similar to br-Z1/dmmpmm(Pollard)/fly(0.710)\n",
      "\t\t16 of 17 (1e-2) similar to SPT15/MA0386.1/Jaspar(0.778)\n",
      "\t\t17 of 17 (1e-1) similar to ZML2(C2C2gata)/col-ZML2-DAP-Seq(GSE60143)/Homer(0.936)\n",
      "\tJob finished\n",
      "\n",
      "treatedF05_normalFVB.filt.bed\n",
      "\n",
      "Selected Options:\n",
      "\tInput file = treatedF05_normalFVB.filt.100bp.fa\n",
      "\tPromoter Set = fasta\n",
      "\tOutput Directory = motifResults/HOMER_DENOVO_treatedF05_normalFVB.filt.bed_vs_GATK.dbSNP.bed\n",
      "\tWill use FASTA files for motif finding\n",
      "\t\tTarget Sequences = treatedF05_normalFVB.filt.100bp.fa\n",
      "\t\tBackground Sequences = GATK.dbSNP.100bp.fa\n",
      "\tWill not search for known motifs\n",
      "\tUsing custom gene IDs for GO analysis\n",
      "\tParsing FASTA format files...\n",
      "\tFound 1662 sequences\n",
      "\tFound 31078 sequences\n",
      "\n",
      "\tProgress: Step4 - removing redundant promoters\n",
      "\n",
      "\tProgress: Step5 - adjusting background sequences for GC/CpG content...\n",
      "\n",
      "\tSequences processed:\n",
      "\t\tAuto detected maximum sequence length of 101 bp\n",
      "\t\t32740 total\n",
      "\n",
      "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
      "\tFreq\tBin\tCount\n",
      "\t0.2\t0\t343\n",
      "\t0.25\t1\t831\n",
      "\t0.3\t2\t2467\n",
      "\t0.35\t3\t4625\n",
      "\t0.4\t4\t6445\n",
      "\t0.45\t5\t6301\n",
      "\t0.5\t6\t5862\n",
      "\t0.6\t7\t5175\n",
      "\t0.7\t8\t597\n",
      "\t0.8\t9\t90\n",
      "\t10\t10\t4\n",
      "\tBin\t# Targets\t# Background\tBackground Weight\n",
      "\t0\t21\t322\t1.220\n",
      "\t1\t52\t779\t1.248\n",
      "\t2\t115\t2352\t0.914\n",
      "\t3\t212\t4413\t0.898\n",
      "\t4\t285\t6160\t0.865\n",
      "\t5\t322\t5979\t1.007\n",
      "\t6\t292\t5570\t0.980\n",
      "\t7\t313\t4862\t1.204\n",
      "\t8\t39\t558\t1.307\n",
      "\t9\t10\t80\t2.337\n",
      "\t10\t1\t3\t6.233\n",
      "\n",
      "\tNormalizing lower order oligos using homer2\n",
      "\n",
      "\tReading input files...\n",
      "\t32740 total sequences read\n",
      "\tAutonormalization: 1-mers (4 total)\n",
      "\t\tA\t28.90%\t28.91%\t1.000\n",
      "\t\tC\t21.10%\t21.09%\t1.000\n",
      "\t\tG\t21.10%\t21.09%\t1.000\n",
      "\t\tT\t28.90%\t28.91%\t1.000\n",
      "\tAutonormalization: 2-mers (16 total)\n",
      "\t\tAA\t8.94%\t8.65%\t1.033\n",
      "\t\tCA\t7.25%\t7.46%\t0.972\n",
      "\t\tGA\t6.28%\t6.41%\t0.979\n",
      "\t\tTA\t6.43%\t6.38%\t1.008\n",
      "\t\tAC\t5.16%\t5.34%\t0.967\n",
      "\t\tCC\t5.37%\t5.29%\t1.015\n",
      "\t\tGC\t4.29%\t4.06%\t1.056\n",
      "\t\tTC\t6.28%\t6.41%\t0.979\n",
      "\t\tAG\t7.37%\t7.43%\t0.993\n",
      "\t\tCG\t1.12%\t0.92%\t1.219\n",
      "\t\tGG\t5.37%\t5.29%\t1.015\n",
      "\t\tTG\t7.25%\t7.46%\t0.972\n",
      "\t\tAT\t7.42%\t7.50%\t0.989\n",
      "\t\tCT\t7.37%\t7.43%\t0.993\n",
      "\t\tGT\t5.16%\t5.34%\t0.967\n",
      "\t\tTT\t8.94%\t8.65%\t1.033\n",
      "\tAutonormalization: 3-mers (64 total)\n",
      "\tNormalization weights can be found in file: motifResults/HOMER_DENOVO_treatedF05_normalFVB.filt.bed_vs_GATK.dbSNP.bed/seq.autonorm.tsv\n",
      "\tConverging on autonormalization solution:\n",
      "\t...............................................................................\n",
      "\tFinal normalization:\tAutonormalization: 1-mers (4 total)\n",
      "\t\tA\t28.90%\t28.90%\t1.000\n",
      "\t\tC\t21.10%\t21.10%\t1.000\n",
      "\t\tG\t21.10%\t21.10%\t1.000\n",
      "\t\tT\t28.90%\t28.90%\t1.000\n",
      "\tAutonormalization: 2-mers (16 total)\n",
      "\t\tAA\t8.94%\t8.94%\t1.000\n",
      "\t\tCA\t7.25%\t7.25%\t1.001\n",
      "\t\tGA\t6.28%\t6.29%\t0.999\n",
      "\t\tTA\t6.43%\t6.42%\t1.002\n",
      "\t\tAC\t5.16%\t5.17%\t0.998\n",
      "\t\tCC\t5.37%\t5.37%\t0.999\n",
      "\t\tGC\t4.29%\t4.28%\t1.001\n",
      "\t\tTC\t6.28%\t6.29%\t0.999\n",
      "\t\tAG\t7.37%\t7.37%\t1.001\n",
      "\t\tCG\t1.12%\t1.11%\t1.004\n",
      "\t\tGG\t5.37%\t5.37%\t0.999\n",
      "\t\tTG\t7.25%\t7.25%\t1.001\n",
      "\t\tAT\t7.42%\t7.43%\t0.999\n",
      "\t\tCT\t7.37%\t7.37%\t1.001\n",
      "\t\tGT\t5.16%\t5.17%\t0.998\n",
      "\t\tTT\t8.94%\t8.94%\t1.000\n",
      "\tAutonormalization: 3-mers (64 total)\n",
      "\n",
      "\tProgress: Step6 - Gene Ontology Enrichment Analysis\n",
      "\tSkipping...\n",
      "\n",
      "\tProgress: Step7 - Known motif enrichment\n",
      "\tSkipping...\n",
      "\n",
      "\tProgress: Step8 - De novo motif finding (HOMER)\n",
      "\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 32506\n",
      "\tAutoadjustment for sequence coverage in background: 1.00x\n",
      "\n",
      "\tOligos: 32506 of 34497 max\n",
      "\tTree  : 66684 of 172485 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 1 mismatches...\n",
      "\n",
      "\tScreening oligos 32506 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t17.47% skipped, 82.53% checked (26826 of 32506), of those checked:\n",
      "\t\t17.47% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 32506 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t17.47% skipped, 82.53% checked (26826 of 32506), of those checked:\n",
      "\t\t0.00% not in target, 32.82% increased p-value, 53.77% high p-value\n",
      "\tReading input files...\n",
      "\t32740 total sequences read\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: TACCTACC... (-18.756)\n",
      "\t\tRound 1: -26.30 TACCTACC T:355.0(19.24%),B:4273.6(13.01%),P:1e-11\n",
      "\t\tRound 2: -26.30 TACCTACC T:355.0(19.24%),B:4273.6(13.01%),P:1e-11\n",
      "\t\t=Final=: -8.79 TACCTACC T:163.0(9.81%),B:2238.5(7.30%),P:1e-3\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: GGGGGGGG... (-16.028)\n",
      "\t\tRound 1: -29.27 GGGGGGGG T:301.0(16.57%),B:3397.5(10.49%),P:1e-12\n",
      "\t\tRound 2: -32.88 GGGGGGGR T:505.0(26.21%),B:6174.2(18.24%),P:1e-14\n",
      "\t\tRound 3: -32.88 GGGGGGGR T:505.0(26.21%),B:6174.2(18.24%),P:1e-14\n",
      "\t\t=Final=: -12.29 GGGGGGGR T:223.0(13.42%),B:3033.1(9.89%),P:1e-5\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: CCATCCAT... (-13.534)\n",
      "\t\tRound 1: -15.09 CCATCCAT T:65.0(3.84%),B:566.2(1.83%),P:1e-6\n",
      "\t\tRound 2: -16.72 CGATCCRT T:58.0(3.43%),B:463.9(1.50%),P:1e-7\n",
      "\t\tRound 3: -16.72 CGATCCRT T:58.0(3.43%),B:463.9(1.50%),P:1e-7\n",
      "\t\t=Final=: -3.26 CGATCCRT T:47.0(2.83%),B:653.6(2.13%),P:1e-1\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: AAGCAAGG... (-12.273)\n",
      "\t\tRound 1: -15.87 AAGCAAGG T:389.0(20.87%),B:5296.4(15.86%),P:1e-6\n",
      "\t\tRound 2: -17.86 AAGCAAGG T:455.0(23.96%),B:6215.7(18.35%),P:1e-7\n",
      "\t\tRound 3: -17.86 AAGCAAGG T:455.0(23.96%),B:6215.7(18.35%),P:1e-7\n",
      "\t\t=Final=: -12.27 AAGCAAGG T:117.0(7.04%),B:1383.6(4.51%),P:1e-5\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: GAAGGAAG... (-12.105)\n",
      "\t\tRound 1: -19.18 GAAGGAAG T:222.0(12.51%),B:2616.3(8.18%),P:1e-8\n",
      "\t\tRound 2: -19.18 GAAGGAAG T:222.0(12.51%),B:2616.3(8.18%),P:1e-8\n",
      "\t\t=Final=: -8.27 GAAGGAAG T:113.0(6.80%),B:1469.2(4.79%),P:1e-3\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: AGAATACA... (-10.892)\n",
      "\t\tRound 1: -14.90 RGAATACV T:234.0(13.14%),B:2977.4(9.25%),P:1e-6\n",
      "\t\tRound 2: -14.90 RGAATACV T:234.0(13.14%),B:2977.4(9.25%),P:1e-6\n",
      "\t\t=Final=: -13.62 RGAATACV T:210.0(12.64%),B:2765.4(9.02%),P:1e-5\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: ATATATAT... (-9.743)\n",
      "\t\tRound 1: -14.48 ATATATAT T:521.0(26.92%),B:7485.6(21.66%),P:1e-6\n",
      "\t\tRound 2: -14.48 ATATATAT T:521.0(26.92%),B:7485.6(21.66%),P:1e-6\n",
      "\t\t=Final=: -3.56 ATATATAT T:89.0(5.35%),B:1325.0(4.32%),P:1e-1\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t8 of 25 Initial Sequence: CAAGGATT... (-9.440)\n",
      "\t\tRound 1: -10.92 CAAGGATT T:15.0(0.90%),B:60.1(0.20%),P:1e-4\n",
      "\t\tRound 2: -10.92 CAAGGATT T:15.0(0.90%),B:60.1(0.20%),P:1e-4\n",
      "\t\t=Final=: -12.14 CAAGGATT T:15.0(0.90%),B:60.1(0.20%),P:1e-5\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\tRemaining seeds don't look promising (After initial 5 motifs, logp -9.194 > -9.221)\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t32740 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing hypergeometric scoring\n",
      "\tChecking enrichment of 8 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: motifResults/HOMER_DENOVO_treatedF05_normalFVB.filt.bed_vs_GATK.dbSNP.bed/homerMotifs.motifs8\n",
      "\n",
      "\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 362240\n",
      "\tAutoadjustment for sequence coverage in background: 1.00x\n",
      "\n",
      "\tOligos: 362240 of 551887 max\n",
      "\tTree  : 780192 of 2759435 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 1 mismatches...\n",
      "\n",
      "\tScreening oligos 362240 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t===============================================================================\n",
      "\t\t70.09% skipped, 29.91% checked (108358 of 362240), of those checked:\n",
      "\t\t70.09% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 362240 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t70.09% skipped, 29.91% checked (108358 of 362240), of those checked:\n",
      "\t\t0.00% not in target, 21.40% increased p-value, 13.18% high p-value\n",
      "\tReading input files...\n",
      "\t32740 total sequences read\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: GTAGGTAAGT... (-24.827)\n",
      "\t\tRound 1: -37.85 GTAGGTAAGT T:44.0(2.61%),B:142.4(0.46%),P:1e-16\n",
      "\t\tRound 2: -37.85 GTAGGTAAGT T:44.0(2.61%),B:142.4(0.46%),P:1e-16\n",
      "\t\t=Final=: -21.31 GTAGGTAAGT T:23.0(1.38%),B:74.0(0.24%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: CCATCCTTCC... (-24.146)\n",
      "\t\tRound 1: -32.51 CCATCCWTCC T:163.0(9.34%),B:1465.9(4.67%),P:1e-14\n",
      "\t\tRound 2: -38.14 CCTTCCWTCC T:186.0(10.59%),B:1645.6(5.22%),P:1e-16\n",
      "\t\tRound 3: -38.14 CCTTCCWTCC T:186.0(10.59%),B:1645.6(5.22%),P:1e-16\n",
      "\t\t=Final=: -8.20 CCTTCCWTCC T:50.0(3.01%),B:530.9(1.73%),P:1e-3\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: TCTATTCTAT... (-17.252)\n",
      "\t\tRound 1: -24.93 TCTATTCTAT T:27.0(1.61%),B:80.8(0.26%),P:1e-10\n",
      "\t\tRound 2: -24.93 TCTATTCTAT T:27.0(1.61%),B:80.8(0.26%),P:1e-10\n",
      "\t\t=Final=: -14.05 TCTATTCTAT T:15.0(0.90%),B:50.7(0.17%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: ACACCCACAC... (-15.366)\n",
      "\t\tRound 1: -17.90 ACACCCACAC T:308.0(16.92%),B:3956.2(12.10%),P:1e-7\n",
      "\t\tRound 2: -17.90 ACACCCACAC T:308.0(16.92%),B:3956.2(12.10%),P:1e-7\n",
      "\t\t=Final=: -6.67 ACACCCACAC T:92.0(5.54%),B:1209.8(3.95%),P:1e-2\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: TCTGCTGAAT... (-14.068)\n",
      "\t\tRound 1: -31.18 TCTGCTGAAT T:29.0(1.73%),B:69.3(0.22%),P:1e-13\n",
      "\t\tRound 2: -33.79 TCTGCTGAAT T:36.0(2.14%),B:104.0(0.34%),P:1e-14\n",
      "\t\tRound 3: -33.79 TCTGCTGAAT T:36.0(2.14%),B:104.0(0.34%),P:1e-14\n",
      "\t\t=Final=: -35.93 TCTGCTGAAT T:36.0(2.17%),B:101.0(0.33%),P:1e-15\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: AAATATATAT... (-13.910)\n",
      "\t\tRound 1: -19.05 AAATATATAT T:397.0(21.25%),B:5249.2(15.73%),P:1e-8\n",
      "\t\tRound 2: -19.23 AAATAYATAT T:502.0(26.08%),B:6852.0(20.03%),P:1e-8\n",
      "\t\tRound 3: -23.00 AAATAYWTAT T:502.0(26.08%),B:6626.9(19.43%),P:1e-9\n",
      "\t\tRound 4: -23.06 AAATAYWWAT T:484.0(25.27%),B:6359.0(18.73%),P:1e-10\n",
      "\t\tRound 5: -23.06 AAATAYWWAT T:484.0(25.27%),B:6359.0(18.73%),P:1e-10\n",
      "\t\t=Final=: -7.12 AAATAYWWAT T:22.0(1.32%),B:181.8(0.59%),P:1e-3\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: GTCAAAAAGA... (-12.354)\n",
      "\t\tRound 1: -21.01 GTCAAAAAGA T:43.0(2.55%),B:243.9(0.79%),P:1e-9\n",
      "\t\tRound 2: -21.01 GTCAAAAAGA T:43.0(2.55%),B:243.9(0.79%),P:1e-9\n",
      "\t\t=Final=: -20.79 GTCAAAAAGA T:41.0(2.47%),B:234.9(0.77%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t8 of 25 Initial Sequence: GGGGGGCGCA... (-11.596)\n",
      "\t\tRound 1: -19.43 GGGGGGCGCA T:14.0(0.84%),B:21.6(0.07%),P:1e-8\n",
      "\t\tRound 2: -19.43 GGGGGGCGCA T:14.0(0.84%),B:21.6(0.07%),P:1e-8\n",
      "\t\t=Final=: -21.42 GGGGGGCGCA T:12.0(0.72%),B:12.9(0.04%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t9 of 25 Initial Sequence: CGTGTCCACT... (-10.755)\n",
      "\t\tRound 1: -16.37 CGTGTCCACT T:11.0(0.66%),B:15.0(0.05%),P:1e-7\n",
      "\t\tRound 2: -16.37 CGTGTCCACT T:11.0(0.66%),B:15.0(0.05%),P:1e-7\n",
      "\t\t=Final=: -18.04 CGTGTCCACT T:11.0(0.66%),B:15.0(0.05%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t10 of 25 Initial Sequence: AAACAGTATT... (-10.585)\n",
      "\t\tRound 1: -22.75 AAACAGTATT T:27.0(1.61%),B:90.7(0.29%),P:1e-9\n",
      "\t\tRound 2: -22.75 AAACAGTATT T:27.0(1.61%),B:90.7(0.29%),P:1e-9\n",
      "\t\t=Final=: -24.55 AAACAGTATT T:28.0(1.68%),B:95.9(0.31%),P:1e-10\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t11 of 25 Initial Sequence: AGATGACTTC... (-10.452)\n",
      "\t\tRound 1: -21.24 AGATGACTTC T:26.0(1.55%),B:90.0(0.29%),P:1e-9\n",
      "\t\tRound 2: -24.68 AGATGACTTC T:47.0(2.79%),B:251.2(0.82%),P:1e-10\n",
      "\t\tRound 3: -25.73 AGATGACTTC T:35.0(2.08%),B:136.5(0.44%),P:1e-11\n",
      "\t\tRound 4: -27.50 AGATGWCTTC T:36.0(2.14%),B:134.3(0.44%),P:1e-11\n",
      "\t\tRound 5: -27.50 AGATGWCTTC T:36.0(2.14%),B:134.3(0.44%),P:1e-11\n",
      "\t\t=Final=: -29.14 AGATGWCTTC T:36.0(2.17%),B:132.1(0.43%),P:1e-12\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t12 of 25 Initial Sequence: TGGCCTAGGC... (-10.307)\n",
      "\t\tRound 1: -13.10 TGGCCTAGGC T:6.0(0.36%),B:2.5(0.01%),P:1e-5\n",
      "\t\tRound 2: -13.10 TGGCCTAGGC T:6.0(0.36%),B:2.5(0.01%),P:1e-5\n",
      "\t\t=Final=: -9.25 TGGCCTAGGC T:4.0(0.24%),B:2.5(0.01%),P:1e-4\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t13 of 25 Initial Sequence: GGTGTTTCCT... (-10.291)\n",
      "\t\tRound 1: -20.67 GGTGTTTCCT T:26.0(1.55%),B:93.1(0.30%),P:1e-8\n",
      "\t\tRound 2: -23.19 GGTGTTTCCT T:40.0(2.38%),B:196.6(0.64%),P:1e-10\n",
      "\t\tRound 3: -27.75 GGTGTTTCCT T:53.0(3.14%),B:283.9(0.92%),P:1e-12\n",
      "\t\tRound 4: -28.73 GGTGTTTCCT T:40.0(2.38%),B:159.2(0.52%),P:1e-12\n",
      "\t\tRound 5: -28.73 GGTGTTTCCT T:40.0(2.38%),B:159.2(0.52%),P:1e-12\n",
      "\t\t=Final=: -30.12 GGTGTTTCCT T:40.0(2.41%),B:158.5(0.52%),P:1e-13\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t14 of 25 Initial Sequence: AAGGCTCCAA... (-9.917)\n",
      "\t\tRound 1: -17.06 AAGGCTCCAA T:58.0(3.43%),B:458.9(1.48%),P:1e-7\n",
      "\t\tRound 2: -17.06 AAGGCTCCAA T:58.0(3.43%),B:458.9(1.48%),P:1e-7\n",
      "\t\t=Final=: -17.71 AAGGCTCCAA T:49.0(2.95%),B:352.4(1.15%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t15 of 25 Initial Sequence: ACAGAGCTCT... (-9.726)\n",
      "\t\tRound 1: -14.34 ACAGAGCTCT T:53.0(3.14%),B:439.9(1.42%),P:1e-6\n",
      "\t\tRound 2: -14.34 ACAGAGCTCT T:53.0(3.14%),B:439.9(1.42%),P:1e-6\n",
      "\t\t=Final=: -12.24 ACAGAGCTCT T:46.0(2.77%),B:395.0(1.29%),P:1e-5\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t16 of 25 Initial Sequence: CTGTGGTTAC... (-10.307)\n",
      "\t\tRound 1: -10.31 CTGTGGTTAC T:5.0(0.30%),B:2.9(0.01%),P:1e-4\n",
      "\t\tRound 2: -10.31 CTGTGGTTAC T:5.0(0.30%),B:2.9(0.01%),P:1e-4\n",
      "\t\t=Final=: -11.89 CTGTGGTTAC T:5.0(0.30%),B:2.9(0.01%),P:1e-5\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t17 of 25 Initial Sequence: GATAAGCAAG... (-9.644)\n",
      "\t\tRound 1: -20.20 GATAAGCAAG T:16.0(0.96%),B:29.2(0.09%),P:1e-8\n",
      "\t\tRound 2: -20.20 GATAAGCAAG T:16.0(0.96%),B:29.2(0.09%),P:1e-8\n",
      "\t\t=Final=: -17.93 GATAAGCAAG T:14.0(0.84%),B:29.2(0.10%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t18 of 25 Initial Sequence: CCTCTGGCTT... (-9.624)\n",
      "\t\tRound 1: -15.83 CCTCTGGCTT T:75.0(4.41%),B:683.2(2.20%),P:1e-6\n",
      "\t\tRound 2: -19.89 CCTCTGGCTT T:52.0(3.08%),B:350.6(1.13%),P:1e-8\n",
      "\t\tRound 3: -19.89 CCTCTGGCTT T:52.0(3.08%),B:350.6(1.13%),P:1e-8\n",
      "\t\t=Final=: -19.04 CCTCTGGCTT T:50.0(3.01%),B:347.4(1.13%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t19 of 25 Initial Sequence: TTAATATCTC... (-9.291)\n",
      "\t\tRound 1: -18.68 TTAATATCTC T:28.0(1.67%),B:121.1(0.39%),P:1e-8\n",
      "\t\tRound 2: -18.76 WTAATATCTC T:27.0(1.61%),B:112.6(0.36%),P:1e-8\n",
      "\t\tRound 3: -18.76 WTAATATCTC T:27.0(1.61%),B:112.6(0.36%),P:1e-8\n",
      "\t\t=Final=: -19.95 WTAATATCTC T:27.0(1.62%),B:112.6(0.37%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t20 of 25 Initial Sequence: CCTTCTGATG... (-9.250)\n",
      "\t\tRound 1: -13.98 CCTTCTGATG T:12.0(0.72%),B:25.5(0.08%),P:1e-6\n",
      "\t\tRound 2: -13.98 CCTTCTGATG T:12.0(0.72%),B:25.5(0.08%),P:1e-6\n",
      "\t\t=Final=: -15.52 CCTTCTGATG T:12.0(0.72%),B:25.5(0.08%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\tRemaining seeds don't look promising (After initial 5 motifs, logp -9.201 > -9.221)\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t32740 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing hypergeometric scoring\n",
      "\tChecking enrichment of 20 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: motifResults/HOMER_DENOVO_treatedF05_normalFVB.filt.bed_vs_GATK.dbSNP.bed/homerMotifs.motifs10\n",
      "\n",
      "\n",
      "\t-blen automatically set to 2\n",
      "\tScanning input files...\n",
      "\tParsing sequences...\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\tTotal number of Oligos: 1282523\n",
      "\tAutoadjustment for sequence coverage in background: 1.00x\n",
      "\n",
      "\tOligos: 1282523 of 2946600 max\n",
      "\tTree  : 3294696 of 14733000 max\n",
      "\tOptimizing memory usage...\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tGlobal Optimization Phase: Looking for enriched oligos with up to 1 mismatches...\n",
      "\n",
      "\tScreening oligos 1282523 (allowing 0 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t89.47% skipped, 10.53% checked (135001 of 1282523), of those checked:\n",
      "\t\t89.47% not in target, 0.00% increased p-value, 0.00% high p-value\n",
      "\n",
      "\tScreening oligos 1282523 (allowing 1 mismatches):\n",
      "\t|0%                                   50%                                  100%|\n",
      "\t================================================================================\n",
      "\t\t89.47% skipped, 10.53% checked (135001 of 1282523), of those checked:\n",
      "\t\t0.00% not in target, 9.63% increased p-value, 0.70% high p-value\n",
      "\tReading input files...\n",
      "\t32740 total sequences read\n",
      "\tCache length = 44721\n",
      "\tUsing hypergeometric scoring\n",
      "\n",
      "\tLocal Optimization Phase:\n",
      "\t1 of 25 Initial Sequence: CCATCCATCCAT... (-22.971)\n",
      "\t\tRound 1: -27.65 CCATCCATCCAT T:43.0(2.55%),B:191.5(0.62%),P:1e-12\n",
      "\t\tRound 2: -27.65 CCATCCATCCAT T:43.0(2.55%),B:191.5(0.62%),P:1e-12\n",
      "\t\t=Final=: -3.24 CCATCCATCCAT T:7.0(0.42%),B:55.4(0.18%),P:1e-1\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t2 of 25 Initial Sequence: TGTAGGTAGGTA... (-21.866)\n",
      "\t\tRound 1: -28.57 TGTAGGTAGGWA T:33.0(1.97%),B:106.3(0.35%),P:1e-12\n",
      "\t\tRound 2: -31.06 GGTAGGAAGGHA T:151.0(8.69%),B:1342.7(4.28%),P:1e-13\n",
      "\t\tRound 3: -33.59 TGTAGGAARGHA T:158.0(9.07%),B:1377.0(4.39%),P:1e-14\n",
      "\t\tRound 4: -33.59 TGTAGGAARGHA T:158.0(9.07%),B:1377.0(4.39%),P:1e-14\n",
      "\t\t=Final=: -11.29 TGTAGGAARGHA T:44.0(2.65%),B:386.2(1.26%),P:1e-4\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t3 of 25 Initial Sequence: AGGAAGGAAGGA... (-152.454)\n",
      "\t\tRound 1: -3.08 SBKACGGNYBTN T:4.0(0.24%),B:14.6(0.05%),P:1e-1\n",
      "\t\tRound 2: -8.47 SGGACGGCCCTB T:6.0(0.36%),B:8.6(0.03%),P:1e-3\n",
      "\t\tRound 3: -10.95 SGGARGGCYCTC T:6.0(0.36%),B:4.2(0.01%),P:1e-4\n",
      "\t\tRound 4: -13.56 GGTAGGGCYTTK T:18.0(1.08%),B:69.8(0.22%),P:1e-5\n",
      "\t\tRound 5: -15.16 GGGAGGGCTCTT T:14.0(0.84%),B:34.0(0.11%),P:1e-6\n",
      "\t\tRound 6: -15.78 GGGDGGGCTCTT T:15.0(0.90%),B:37.9(0.12%),P:1e-6\n",
      "\t\tRound 7: -15.88 GGGGGGGCTCTT T:11.0(0.66%),B:15.1(0.05%),P:1e-6\n",
      "\t\tRound 8: -15.88 GGGGGGGCTCTT T:11.0(0.66%),B:15.1(0.05%),P:1e-6\n",
      "\t\t=Final=: -17.54 GGGGGGGCTCTT T:11.0(0.66%),B:15.1(0.05%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t4 of 25 Initial Sequence: TCTATTCTATCC... (-20.236)\n",
      "\t\tRound 1: -28.94 TCTATTCTATYC T:22.0(1.32%),B:37.3(0.12%),P:1e-12\n",
      "\t\tRound 2: -31.38 TCTATTCTATCC T:23.0(1.37%),B:36.9(0.12%),P:1e-13\n",
      "\t\tRound 3: -31.38 TCTATTCTATCC T:23.0(1.37%),B:36.9(0.12%),P:1e-13\n",
      "\t\t=Final=: -21.42 TCTATTCTATCC T:12.0(0.72%),B:12.7(0.04%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t5 of 25 Initial Sequence: CTCACACACACA... (-17.081)\n",
      "\t\tRound 1: -22.58 CTCACACACACA T:262.0(14.59%),B:3076.2(9.54%),P:1e-9\n",
      "\t\tRound 2: -23.65 CTCACACACACA T:250.0(13.97%),B:2871.6(8.94%),P:1e-10\n",
      "\t\tRound 3: -23.65 CTCACACACACA T:250.0(13.97%),B:2871.6(8.94%),P:1e-10\n",
      "\t\t=Final=: -7.60 CTCACACACACA T:24.0(1.44%),B:198.7(0.65%),P:1e-3\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t6 of 25 Initial Sequence: CTATTTGAGGAG... (-14.845)\n",
      "\t\tRound 1: -16.08 CTATTTGAGGAG T:51.0(3.02%),B:388.2(1.26%),P:1e-6\n",
      "\t\tRound 2: -22.25 CTATTTGAGGAG T:88.0(5.16%),B:724.1(2.33%),P:1e-9\n",
      "\t\tRound 3: -22.95 CTATTTGAGGAG T:71.0(4.18%),B:518.7(1.68%),P:1e-9\n",
      "\t\tRound 4: -30.38 CTATTTGARGAG T:53.0(3.14%),B:262.6(0.85%),P:1e-13\n",
      "\t\tRound 5: -30.38 CTATTTGARGAG T:53.0(3.14%),B:262.6(0.85%),P:1e-13\n",
      "\t\t=Final=: -32.29 CTATTTGARGAG T:53.0(3.19%),B:255.5(0.83%),P:1e-14\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t7 of 25 Initial Sequence: ATATATATATGT... (-14.292)\n",
      "\t\tRound 1: -17.13 ATATATATATGT T:332.0(18.11%),B:4362.2(13.26%),P:1e-7\n",
      "\t\tRound 2: -20.59 ATMTATATATGT T:367.0(19.82%),B:4712.2(14.24%),P:1e-8\n",
      "\t\tRound 3: -22.44 ATATATWTRTAT T:457.0(24.05%),B:5970.4(17.69%),P:1e-9\n",
      "\t\tRound 4: -23.55 ATATATWTDTWT T:506.0(26.25%),B:6657.4(19.52%),P:1e-10\n",
      "\t\tRound 5: -23.55 ATATATWTDTWT T:506.0(26.25%),B:6657.4(19.52%),P:1e-10\n",
      "\t\t=Final=: -3.54 ATATATWTDTWT T:12.0(0.72%),B:115.0(0.37%),P:1e-1\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t8 of 25 Initial Sequence: GACAACACATTT... (-12.952)\n",
      "\t\tRound 1: -18.18 GACAACACATTT T:14.0(0.84%),B:24.9(0.08%),P:1e-7\n",
      "\t\tRound 2: -18.18 GACAACACATTT T:14.0(0.84%),B:24.9(0.08%),P:1e-7\n",
      "\t\t=Final=: -19.78 GACAACACATTT T:14.0(0.84%),B:24.9(0.08%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t9 of 25 Initial Sequence: GGGTCCTTCCTT... (-12.572)\n",
      "\t\tRound 1: -16.03 GGGTCCTTCCTT T:9.0(0.54%),B:7.5(0.02%),P:1e-6\n",
      "\t\tRound 2: -16.03 GGGTCCTTCCTT T:9.0(0.54%),B:7.5(0.02%),P:1e-6\n",
      "\t\t=Final=: -17.72 GGGTCCTTCCTT T:9.0(0.54%),B:7.5(0.02%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t10 of 25 Initial Sequence: GTGGCTGATGTG... (-11.888)\n",
      "\t\tRound 1: -19.40 GTGGCTGATGTG T:11.0(0.66%),B:9.2(0.03%),P:1e-8\n",
      "\t\tRound 2: -19.40 GTGGCTGATGTG T:11.0(0.66%),B:9.2(0.03%),P:1e-8\n",
      "\t\t=Final=: -21.83 GTGGCTGATGTG T:11.0(0.66%),B:8.5(0.03%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t11 of 25 Initial Sequence: TCTGCTGAATGT... (-11.351)\n",
      "\t\tRound 1: -25.49 TCTGCTGAATGT T:26.0(1.55%),B:71.6(0.23%),P:1e-11\n",
      "\t\tRound 2: -25.49 TCTGCTGAATGT T:26.0(1.55%),B:71.6(0.23%),P:1e-11\n",
      "\t\t=Final=: -27.17 TCTGCTGAATGT T:26.0(1.56%),B:70.7(0.23%),P:1e-11\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t12 of 25 Initial Sequence: TTGGTGTTTCCT... (-11.258)\n",
      "\t\tRound 1: -20.04 TTGGTGTTTCCT T:9.0(0.54%),B:3.6(0.01%),P:1e-8\n",
      "\t\tRound 2: -20.04 TTGGTGTTTCCT T:9.0(0.54%),B:3.6(0.01%),P:1e-8\n",
      "\t\t=Final=: -21.61 TTGGTGTTTCCT T:8.0(0.48%),B:1.7(0.01%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t13 of 25 Initial Sequence: TGATTACTGTCA... (-10.307)\n",
      "\t\tRound 1: -11.87 TGATTACTGTCA T:5.0(0.30%),B:0.0(0.00%),P:1e-5\n",
      "\t\tRound 2: -11.87 TGATTACTGTCA T:5.0(0.30%),B:0.0(0.00%),P:1e-5\n",
      "\t\t=Final=: -14.85 TGATTACTGTCA T:5.0(0.30%),B:0.0(0.00%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t14 of 25 Initial Sequence: GCACACAGTACT... (-10.307)\n",
      "\t\tRound 1: -11.89 GCACACAGTACT T:6.0(0.36%),B:3.3(0.01%),P:1e-5\n",
      "\t\tRound 2: -13.61 RCASACAGCACT T:8.0(0.48%),B:7.4(0.02%),P:1e-5\n",
      "\t\tRound 3: -16.95 GCAGACAGCACT T:15.0(0.90%),B:33.1(0.11%),P:1e-7\n",
      "\t\tRound 4: -16.95 GCAGACAGCACT T:15.0(0.90%),B:33.1(0.11%),P:1e-7\n",
      "\t\t=Final=: -18.80 GCAGACAGCACT T:15.0(0.90%),B:32.2(0.10%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t15 of 25 Initial Sequence: GACTGCAGGGCC... (-10.307)\n",
      "\t\tRound 1: -18.70 GACTGCAGGGCC T:11.0(0.66%),B:10.0(0.03%),P:1e-8\n",
      "\t\tRound 2: -20.16 GACTGCAGGGCC T:11.0(0.66%),B:8.9(0.03%),P:1e-8\n",
      "\t\tRound 3: -20.16 GACTGCAGGGCC T:11.0(0.66%),B:8.9(0.03%),P:1e-8\n",
      "\t\t=Final=: -21.83 GACTGCAGGGCC T:11.0(0.66%),B:8.1(0.03%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t16 of 25 Initial Sequence: GTGTGTGTGAGA... (-10.307)\n",
      "\t\tRound 1: -7.98 GTGTGTGTGAGA T:191.0(10.86%),B:2670.6(8.34%),P:1e-3\n",
      "\t\tRound 2: -7.98 GTGTGTGTGAGA T:191.0(10.86%),B:2670.6(8.34%),P:1e-3\n",
      "\t\t=Final=: -3.83 GTGTGTGTGAGA T:37.0(2.23%),B:471.3(1.54%),P:1e-1\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t17 of 25 Initial Sequence: ATGTCTTTATAT... (-10.291)\n",
      "\t\tRound 1: -16.32 ATGTCTTTATAT T:37.0(2.20%),B:227.4(0.74%),P:1e-7\n",
      "\t\tRound 2: -20.63 ATGTCTTTATAT T:31.0(1.85%),B:134.8(0.44%),P:1e-8\n",
      "\t\tRound 3: -20.63 ATGTCTTTATAT T:31.0(1.85%),B:134.8(0.44%),P:1e-8\n",
      "\t\t=Final=: -19.33 ATGTCTTTATAT T:29.0(1.74%),B:133.6(0.44%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t18 of 25 Initial Sequence: ATCAAACAGTTG... (-10.183)\n",
      "\t\tRound 1: -14.33 ATCAAACAGTTG T:22.0(1.32%),B:98.9(0.32%),P:1e-6\n",
      "\t\tRound 2: -14.33 ATCAAACAGTTG T:22.0(1.32%),B:98.9(0.32%),P:1e-6\n",
      "\t\t=Final=: -15.48 ATCAAACAGTTG T:22.0(1.32%),B:98.9(0.32%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t19 of 25 Initial Sequence: CTGAGGTCATAT... (-10.166)\n",
      "\t\tRound 1: -17.82 CTGAGGTCATAT T:7.0(0.42%),B:0.0(0.00%),P:1e-7\n",
      "\t\tRound 2: -21.49 CAGAAATCATAC T:21.0(1.26%),B:54.0(0.18%),P:1e-9\n",
      "\t\tRound 3: -21.49 CAGAAATCATAC T:21.0(1.26%),B:54.0(0.18%),P:1e-9\n",
      "\t\t=Final=: -22.93 CAGAAATCATAC T:21.0(1.26%),B:54.0(0.18%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t20 of 25 Initial Sequence: CAAAAATCAAGG... (-10.107)\n",
      "\t\tRound 1: -17.79 CAAAAATCAAGG T:14.0(0.84%),B:25.1(0.08%),P:1e-7\n",
      "\t\tRound 2: -17.79 CAAAAATCAAGG T:14.0(0.84%),B:25.1(0.08%),P:1e-7\n",
      "\t\t=Final=: -19.38 CAAAAATCAAGG T:14.0(0.84%),B:25.1(0.08%),P:1e-8\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t21 of 25 Initial Sequence: CAAATCAAGGCA... (-10.122)\n",
      "\t\tRound 1: -17.82 CAAATCAAGGCA T:7.0(0.42%),B:0.6(0.00%),P:1e-7\n",
      "\t\tRound 2: -17.82 CAAATCAAGGCA T:7.0(0.42%),B:0.6(0.00%),P:1e-7\n",
      "\t\t=Final=: -20.79 CAAATCAAGGCA T:7.0(0.42%),B:0.6(0.00%),P:1e-9\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t22 of 25 Initial Sequence: TGGACTCTGGGA... (-9.762)\n",
      "\t\tRound 1: -20.79 TGGACTCTGGGA T:8.0(0.48%),B:0.7(0.00%),P:1e-9\n",
      "\t\tRound 2: -20.79 TGGACTCTGGGA T:8.0(0.48%),B:0.7(0.00%),P:1e-9\n",
      "\t\t=Final=: -23.76 TGGACTCTGGGA T:8.0(0.48%),B:0.7(0.00%),P:1e-10\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t23 of 25 Initial Sequence: TGATTTTACCCT... (-9.533)\n",
      "\t\tRound 1: -13.98 TGATTTTACCCT T:12.0(0.72%),B:25.4(0.08%),P:1e-6\n",
      "\t\tRound 2: -13.98 TGATTTTACCCT T:12.0(0.72%),B:25.4(0.08%),P:1e-6\n",
      "\t\t=Final=: -15.52 TGATTTTACCCT T:12.0(0.72%),B:25.4(0.08%),P:1e-6\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t24 of 25 Initial Sequence: TGTTTAAGAGTG... (-9.533)\n",
      "\t\tRound 1: -14.84 TGTTTAAGAGTG T:6.0(0.36%),B:0.0(0.00%),P:1e-6\n",
      "\t\tRound 2: -14.84 TGTTTAAGAGTG T:6.0(0.36%),B:0.0(0.00%),P:1e-6\n",
      "\t\t=Final=: -17.82 TGTTTAAGAGTG T:6.0(0.36%),B:0.0(0.00%),P:1e-7\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\t25 of 25 Initial Sequence: CACCCCTCCCAC... (-9.458)\n",
      "\t\tRound 1: -18.75 CACCCCTCCCAC T:8.0(0.48%),B:2.2(0.01%),P:1e-8\n",
      "\t\tRound 2: -19.19 CACCCCTWCCAM T:13.0(0.78%),B:17.2(0.06%),P:1e-8\n",
      "\t\tRound 3: -23.24 CACCCCTTCCAC T:28.0(1.67%),B:95.5(0.31%),P:1e-10\n",
      "\t\tRound 4: -23.24 CACCCCTTCCAC T:28.0(1.67%),B:95.5(0.31%),P:1e-10\n",
      "\t\t=Final=: -23.04 CACCCCTTCCAC T:27.0(1.62%),B:95.5(0.31%),P:1e-10\n",
      "\t\tPerforming exhaustive masking of motif...\n",
      "\t\tReprioritizing potential motifs...\n",
      "\n",
      "\tFinalizing Enrichment Statistics (new in v3.4)\n",
      "\tReading input files...\n",
      "\t32740 total sequences read\n",
      "\tCache length = 11180\n",
      "\tUsing hypergeometric scoring\n",
      "\tChecking enrichment of 25 motif(s)\n",
      "\t|0%                                    50%                                  100%|\n",
      "\t=================================================================================\n",
      "\tOutput in file: motifResults/HOMER_DENOVO_treatedF05_normalFVB.filt.bed_vs_GATK.dbSNP.bed/homerMotifs.motifs12\n",
      "\n",
      "\t(Motifs in homer2 format)\n",
      "\tDetermining similar motifs... 53 reduced to 32 motifs\n",
      "\tOutputing HTML and sequence logos for motif comparison...\n",
      "\tChecking de novo motifs against known motifs...\n",
      "\tFormatting HTML page...\n",
      "\t\t1 of 32 (1e-15) similar to SWI5/Literature(Harbison)/Yeast(0.751)\n",
      "\t\t2 of 32 (1e-14) similar to Mef2a(MADS)/HL1-Mef2a.biotin-ChIP-Seq(GSE21529)/Homer(0.750)\n",
      "\t\t3 of 32 (1e-12) similar to YAP5(MacIsaac)/Yeast(0.704)\n",
      "\t\t4 of 32 (1e-12) similar to REL/MA0101.1/Jaspar(0.816)\n",
      "\t\t5 of 32 (1e-10) similar to PB0141.1_Isgf3g_2/Jaspar(0.736)\n",
      "\t\t6 of 32 (1e-10) similar to ZNF143|STAF(Zf)/CUTLL-ZNF143-ChIP-Seq(GSE29600)/Homer(0.717)\n",
      "\t\t7 of 32 (1e-9) similar to Maz(Zf)/HepG2-Maz-ChIP-Seq(GSE31477)/Homer(0.724)\n",
      "\t\t8 of 32 (1e-9) similar to SOK2/SOK2_BUT14/4-SUT1(Harbison)/Yeast(0.659)\n",
      "\t\t9 of 32 (1e-9) similar to hb/dmmpmm(Bergman)/fly(0.648)\n",
      "\t\t10 of 32 (1e-9) similar to MYB3/MA1038.1/Jaspar(0.798)\n",
      "\t\t11 of 32 (1e-9) similar to Syncrip(RRM)/Xenopus_tropicalis-RNCMPT00281-PBM/HughesRNA(0.757)\n",
      "\t\t12 of 32 (1e-8) similar to Maz(Zf)/HepG2-Maz-ChIP-Seq(GSE31477)/Homer(0.786)\n",
      "\t\t13 of 32 (1e-8) similar to twi/dmmpmm(Down)/fly(0.693)\n",
      "\t\t14 of 32 (1e-8) similar to PB0126.1_Gata5_2/Jaspar(0.648)\n",
      "\t\t15 of 32 (1e-8) similar to SPT23/SPT23_YPD/[](Harbison)/Yeast(0.773)\n",
      "\t\t16 of 32 (1e-8) similar to br-Z4/dmmpmm(Bergman)/fly(0.674)\n",
      "\t\t17 of 32 (1e-8) similar to Unknown2/Drosophila-Promoters/Homer(0.678)\n",
      "\t\t18 of 32 (1e-8) similar to NRG1/MA0347.1/Jaspar(0.709)\n",
      "\t\t19 of 32 (1e-8) similar to Hand1::Tcf3/MA0092.1/Jaspar(0.754)\n",
      "\t\t20 of 32 (1e-7) similar to FMR1(KH)/Homo_sapiens-RNCMPT00016-PBM/HughesRNA(0.719)\n",
      "\t\t21 of 32 (1e-7) similar to Ct/dmmpmm(Noyes_hd)/fly(0.667)\n",
      "\t\t22 of 32 (1e-6) similar to AT1G76880(Trihelix)/col-AT1G76880-DAP-Seq(GSE60143)/Homer(0.704)\n",
      "\t\t23 of 32 (1e-6) similar to Tb_0230(RRM)/Trypanosoma_brucei-RNCMPT00230-PBM/HughesRNA(0.717)\n",
      "\t\t24 of 32 (1e-6) similar to achi/MA0207.1/Jaspar(0.679)\n",
      "\t\t25 of 32 (1e-5) similar to PB0099.1_Zfp691_1/Jaspar(0.682)\n",
      "\t\t26 of 32 (1e-5) similar to HAT1(Homeobox)/col-HAT1-DAP-Seq(GSE60143)/Homer(0.743)\n",
      "\t\t27 of 32 (1e-4) similar to RUNX-AML(Runt)/CD4+-PolII-ChIP-Seq(Barski_et_al.)/Homer(0.877)\n",
      "\t\t28 of 32 (1e-4) similar to ZNF711(Zf)/SHSY5Y-ZNF711-ChIP-Seq(GSE20673)/Homer(0.754)\n",
      "\t\t29 of 32 (1e-3) similar to RBM5(Znf)/Homo_sapiens-RNCMPT00055-PBM/HughesRNA(0.877)\n",
      "\t\t30 of 32 (1e-3) similar to SeqBias: CA-repeat(0.904)\n",
      "\t\t31 of 32 (1e-3) similar to br-Z1/dmmpmm(SeSiMCMC)/fly(0.721)\n",
      "\t\t32 of 32 (1e-1) similar to unc-86/MA0926.1/Jaspar(0.679)\n",
      "\tJob finished\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# For target sets we use pairwise comparions from the cancer pipeline and also inverting cancer and normal i.e.:\n",
    "#1) F03 VS FVB\n",
    "#2) F05 VS FVB\n",
    "#3) FVB VS F03\n",
    "#4) FVB VS F05\n",
    "for target_filename in glob.glob('*.filt.bed'):\n",
    "    print target_filename\n",
    "    scan_denovo_homer(target_filename,bg_filename,window_size)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\r\n",
      "\tProgram will find de novo and known motifs in a gene list\r\n",
      "\r\n",
      "\t\tUsage:  findMotifs.pl <input list> <promoter set> <output directory> [additoinal options]\r\n",
      "\r\n",
      "\t\texample: findMotifs.pl genelist.txt mouse motifResults/ -len 10\r\n",
      "\r\n",
      "\t\tFASTA example: findMotifs.pl targets.fa fasta motifResults/ -fasta background.fa\r\n",
      "\r\n",
      "\tAvailable Promoter Sets: Add custom promoters sets with loadPromoters.pl\r\n",
      "\r\n",
      "\t\tTry typing \"perl /gcdata/gcproj/Luca/HOMER/.//configureHomer.pl -list\" to see available promoter sets\r\n",
      "\t\tTyping \"perl /gcdata/gcproj/Luca/HOMER/.//configureHomer.pl -install NNN\" to install promoter set NNN\r\n",
      "\r\n",
      "\tBasic options:\r\n",
      "\t\t-len <#>[,<#>,<#>...] (motif length, default=8,10,12) [NOTE: values greater 12 may cause the program\r\n",
      "\t\t\tto run out of memmory - in these cases decrease the number of sequences analyzed]\r\n",
      "\t\t-bg <background file> (ids to use as background, default: all genes)\r\n",
      "\t\t-start <#> (offset from TSS, default=-300) [max=based on Promoter Set]\r\n",
      "\t\t-end <#> (offset from TSS, default=50) [max=based on Promoter Set]\r\n",
      "\t\t-rna (output RNA motif logos and compare to RNA motif database, automatically sets -norevopp)\r\n",
      "\t\t-mask/-nomask (use/don't use repeatmasked files, default: -mask)\r\n",
      "\t\t-S <#> (Number of motifs to optimize, default: 25)\r\n",
      "\t\t-mis <#> (global optimization: searches for strings with # mismatches, default: 1)\r\n",
      "\t\t-noconvert (will not worry about converting input files into unigene ids)\r\n",
      "\t\t-norevopp (do not search the reverse strand for motifs)\r\n",
      "\t\t-nomotif (don't search for de novo motif enrichment)\r\n",
      "\r\n",
      "\tScanning sequence for motifs\r\n",
      "\t\t-find <motif file> (This will cause the program to only scan for motifs)\r\n",
      "\r\n",
      "\tIncluding Enhancers - peak files of enhancer location, peak ID should be gene ID\r\n",
      "\t\t-enhancers <peak file> <genome verion>\r\n",
      "\t\t\t(enhancers to include in search space, peaks/sequences should be named with a gene ID\r\n",
      "\t\t\tIf multiple enhancers per gene, use the same gene ID, and all will be included)\r\n",
      "\t\t-enhancersOnly (do not include promoter sequence in motif search)\r\n",
      "\r\n",
      "\tFASTA files: If you prefer to use your own fasta files, place target sequences and \r\n",
      "\t\tbackground sequences in two separate FASTA formated files (must have unique identifiers)\r\n",
      "\t\tTarget File - use in place of <input list> (i.e. the first argument)\r\n",
      "\t\tBackground File - after output directory (with additional options) use the argument:\r\n",
      "\t\t\t-fastaBg <background fasta file> (This is recommended for fasta based analysis)\r\n",
      "\t\tIn place of the promoter set use \"fasta\", or any valid set (this parameter is ignored)\r\n",
      "\t\tWhen finding motifs [-find], only the target file with be searched)\r\n",
      "\t\t\t-chopify (chops up background regions to match size of target regions)\r\n",
      "\t\t\t\ti.e. if background is a full genome or all mRNAs\r\n",
      "\r\n",
      "\tKnown Motif Options/Visualization:\r\n",
      "\t\t-mset <vertebrates|insects|worms|plants|yeast|all> (check against motif collects, default: auto)\r\n",
      "\t\t-basic (don't check de novo motifs for similarity to known motifs)\r\n",
      "\t\t-bits (scale sequence logos by information content, default: doesn't scale)\r\n",
      "\t\t-nocheck (don't check for similarity between novo motif motifs and known motifs)\r\n",
      "\t\t-mcheck <motif file> (known motifs to check against de novo motifs,\r\n",
      "\t\t-noknown (don't search for known motif enrichment, default: -known)\r\n",
      "\t\t-mknown <motif file> (known motifs to check for enrichment,\r\n",
      "\t\t-nofacts (omit humor)\r\n",
      "\t\t-seqlogo (uses weblogo/seqlogo/ghostscript to visualize motifs, default uses SVG)\r\n",
      "\r\n",
      "\tAdvanced options:\r\n",
      "\t\t-b (use binomial distribution to calculate p-values, hypergeometric is default)\r\n",
      "\t\t-nogo (don't search for gene ontology enrichment)\r\n",
      "\t\t-humanGO (Convert IDs to human for GO analysis)\r\n",
      "\t\t-noweight (no CG correction)\r\n",
      "\t\t-noredun (Don't remove predetermined redundant promoters/sequences)\r\n",
      "\t\t-g (input file is a group file, i.e. 1st column = id, 2nd = 0 or 1 [1=target,0=back])\r\n",
      "\t\t-cpg (use CpG% instead of GC% for sequence normalization)\r\n",
      "\t\t-rand (randomize labels for target and backgound sequences)\r\n",
      "\t\t-maskMotif <motif file 1> [motif file 2] ... (motifs to mask before motif finding)\r\n",
      "\t\t-opt <motif file 1> [motif file 2] ... (motifs to optimize/change length)\r\n",
      "\t\t-peaks (will produce peak file of promoters to use with findMotifsGenome.pl)\r\n",
      "\t\t-nowarn (no warnings)\r\n",
      "\t\t-keepFiles (don't delete temporary files)\r\n",
      "\t\t-dumpFasta (create target.fa and background.fa files)\r\n",
      "\t\t-min <#> (remove sequences shorter than #, default: 0)\r\n",
      "\t\t-max <#> (remove sequences longer than #, default: 1e10)\r\n",
      "\t\t-reuse (rerun homer using old seq files etc. with new options\r\n",
      "\t\t\t  and ignores input list, organism)\r\n",
      "\t\t-fdr <#> (Calculate empirical FDR for de novo discovery #=number of randomizations)\r\n",
      "\r\n",
      "\thomer2 specific options:\r\n",
      "\t\t-homer2 (use homer2 instead of original homer, default)\r\n",
      "\t\t-nlen <#> (length of lower-order oligos to normalize - general sequences, default: 3)\r\n",
      "\t\t\t-nmax <#> (Max normalization iterations, default: 160)\r\n",
      "\t\t\t-neutral (weight sequences to neutral frequencies, i.e. 25%, 6.25%, etc.)\r\n",
      "\t\t-olen <#> (lower-order oligo normalization for oligo table, use if -nlen isn't working well)\r\n",
      "\t\t-p <#> (Number of processors to use, default: 1)\r\n",
      "\t\t-e <#> (Maximum expected motif instance per bp in random sequence, default: 0.01)\r\n",
      "\t\t-cache <#> (size in MB for statistics cache, default: 500)\r\n",
      "\t\t-quickMask (skip full masking after finding motifs, similar to original homer)\r\n",
      "\t\t-homer1 (to force the use of the original homer)\r\n",
      "\t\t-minlp <#> (stop looking for motifs when seed logp score gets above #, default: -10)\r\n",
      "\r\n",
      "\tOriginal homer specific options:\r\n",
      "\t\t-homer1 (to force the use of the original homer)\r\n",
      "\t\t-depth [low|med|high|allnight] (time spent on local optimization default: med)\r\n",
      "\r\n"
     ]
    }
   ],
   "source": [
    "!findMotifs.pl"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python [default]",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
